client/src/components/SVGIconSprite/index.tsx

```diff
@@ -0,0 +1,20 @@
+`;
+
+const SVGIconSprite: React.FunctionComponent = () => {
+  const spritePlaceholderRef = React.useRef<HTMLDivElement | null>();
+
+  React.useEffect(() => {
+    if (spritePlaceholderRef.current) {
+      spritePlaceholderRef.current.innerHTML = spriteHtml;
+    }
+  }, [spritePlaceholderRef]);
```

Comment:
I am not sure of how this component should work, but there are better ways to include inline HTML inside your component:
https://fr.reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml

```ts
function createMarkup() {
  return {__html: spritHtml};
}
```
```tsx
<div dangerouslySetInnerHTML={createMarkup()} />
```

But in this case, I might be missing something, but why not just create a component from the svg markup ? 

```tsx
const SVGIconSprite: React.FunctionComponent = () => {
  return (
      <svg><defs><symbol id="icon-angle-double-left" viewBox="0 0 448 512"><path fill="currentColor" d="M223.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L319.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L393.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34zm-192 34l136 136c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9L127.9 256l96.4-96.4c9.4-9.4 9.4-24.6 0-33.9L201.7 103c-9.4-9.4-24.6-9.4-33.9 0l-136 136c-9.5 9.4-9.5 24.6-.1 34z"></path></symbol><symbol id="icon-angle-double-right" viewBox="0 0 448 512"><path fill="currentColor" d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34zm192-34l-136-136c-9.4-9.4-24.6-9.4-33.9 0l-22.6 22.6c-9.4 9.4-9.4 24.6 0 33.9l96.4 96.4-96.4 96.4c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l136-136c9.4-9.2 9.4-24.4 0-33.8z"></path></symbol><symbol id="icon-arrow-down-big" viewBox="0 0 16 16"><path d="M14.969 6.281c0 0.313-0.125 0.594-0.344 0.813l-5.813 5.813c-0.219 0.219-0.5 0.344-0.813 0.344s-0.594-0.125-0.813-0.344l-5.813-5.813c-0.219-0.219-0.344-0.5-0.344-0.813s0.125-0.594 0.344-0.813l0.656-0.656c0.25-0.219 0.531-0.344 0.813-0.344 0.344 0 0.594 0.125 0.813 0.344l4.344 4.344 4.344-4.344c0.219-0.219 0.469-0.344 0.813-0.344 0.281 0 0.563 0.125 0.813 0.344l0.656 0.656c0.219 0.25 0.344 0.5 0.344 0.813z"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="M12.469 6.563c0 0.094-0.031 0.156-0.094 0.219l-4.156 4.156c-0.063 0.063-0.156 0.094-0.219 0.094s-0.156-0.031-0.219-0.094l-4.156-4.156c-0.063-0.063-0.094-0.125-0.094-0.219 0-0.063 0.031-0.125 0.094-0.188l0.469-0.469c0.031-0.031 0.125-0.063 0.188-0.063 0.094 0 0.156 0.031 0.219 0.063l3.5 3.531 3.5-3.531c0.063-0.031 0.125-0.063 0.219-0.063 0.063 0 0.156 0.031 0.188 0.063l0.469 0.469c0.063 0.063 0.094 0.125 0.094 0.188z"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="M10.75 4.344c0 0.094-0.031 0.156-0.094 0.219l-3.5 3.5 3.5 3.531c0.063 0.031 0.094 0.125 0.094 0.188 0 0.094-0.031 0.156-0.094 0.219l-0.438 0.438c-0.063 0.063-0.156 0.094-0.219 0.094s-0.156-0.031-0.219-0.094l-4.156-4.156c-0.063-0.063-0.094-0.125-0.094-0.219 0-0.063 0.031-0.125 0.094-0.188l4.156-4.156c0.063-0.063 0.156-0.094 0.219-0.094s0.156 0.031 0.219 0.094l0.438 0.438c0.063 0.063 0.094 0.125 0.094 0.188z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="M10.469 8.063c0 0.094-0.031 0.156-0.094 0.219l-4.156 4.156c-0.063 0.063-0.156 0.094-0.219 0.094s-0.156-0.031-0.219-0.094l-0.438-0.438c-0.063-0.063-0.094-0.125-0.094-0.219 0-0.063 0.031-0.156 0.094-0.188l3.5-3.531-3.5-3.5c-0.063-0.063-0.094-0.125-0.094-0.219 0-0.063 0.031-0.125 0.094-0.188l0.438-0.438c0.063-0.063 0.156-0.094 0.219-0.094s0.156 0.031 0.219 0.094l4.156 4.156c0.063 0.063 0.094 0.125 0.094 0.188z"></path></symbol><symbol id="icon-arrow-up-big" viewBox="0 0 16 16"><path d="M14.969 10.844c0 0.344-0.125 0.594-0.344 0.813l-0.656 0.688c-0.25 0.219-0.5 0.313-0.813 0.313-0.344 0-0.594-0.094-0.813-0.313l-4.344-4.344-4.344 4.344c-0.219 0.219-0.469 0.313-0.813 0.313-0.313 0-0.563-0.094-0.781-0.313l-0.688-0.688c-0.219-0.219-0.344-0.469-0.344-0.813 0-0.313 0.125-0.563 0.344-0.813l5.813-5.813c0.219-0.219 0.5-0.313 0.813-0.313s0.594 0.094 0.813 0.313l5.813 5.813c0.219 0.25 0.344 0.5 0.344 0.813z"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="M12.469 10.063c0 0.094-0.031 0.156-0.094 0.219l-0.469 0.438c-0.031 0.063-0.125 0.094-0.188 0.094-0.094 0-0.156-0.031-0.219-0.094l-3.5-3.5-3.5 3.5c-0.063 0.063-0.125 0.094-0.219 0.094-0.063 0-0.156-0.031-0.188-0.094l-0.469-0.438c-0.063-0.063-0.094-0.125-0.094-0.219 0-0.063 0.031-0.125 0.094-0.188l4.156-4.156c0.063-0.063 0.156-0.094 0.219-0.094s0.156 0.031 0.219 0.094l4.156 4.156c0.063 0.063 0.094 0.125 0.094 0.188z"></path></symbol><symbol id="icon-arrows-up-down" viewBox="0 0 16 16"><path d="M10.499 3.468c0 0.137-0.027 0.275-0.137 0.357-0.11 0.11-0.22 0.165-0.357 0.165h-0.989v8.020h0.989c0.137 0 0.247 0.055 0.357 0.165 0.11 0.082 0.137 0.22 0.137 0.357s-0.027 0.247-0.137 0.33l-2.005 2.032c-0.11 0.082-0.22 0.137-0.357 0.137s-0.247-0.055-0.357-0.137l-2.005-2.032c-0.11-0.082-0.137-0.192-0.137-0.33s0.027-0.275 0.137-0.357c0.11-0.11 0.22-0.165 0.357-0.165h0.989v-8.020h-0.989c-0.137 0-0.247-0.055-0.357-0.165-0.11-0.082-0.137-0.22-0.137-0.357s0.027-0.247 0.137-0.33l2.005-2.032c0.11-0.082 0.22-0.137 0.357-0.137s0.247 0.055 0.357 0.137l2.005 2.032c0.11 0.082 0.137 0.192 0.137 0.33z"></path></symbol><symbol id="icon-bin" viewBox="0 0 16 16"><path d="M6.281 6.563v5.156c0 0.094-0.031 0.156-0.063 0.188-0.063 0.063-0.125 0.094-0.219 0.094h-0.563c-0.094 0-0.156-0.031-0.219-0.094-0.063-0.031-0.063-0.094-0.063-0.188v-5.156c0-0.063 0-0.156 0.063-0.188 0.063-0.063 0.125-0.094 0.219-0.094h0.563c0.094 0 0.156 0.031 0.219 0.094 0.031 0.031 0.063 0.125 0.063 0.188zM8.563 6.563v5.156c0 0.094-0.031 0.156-0.063 0.188-0.063 0.063-0.125 0.094-0.219 0.094h-0.563c-0.094 0-0.156-0.031-0.219-0.094-0.031-0.031-0.063-0.094-0.063-0.188v-5.156c0-0.063 0.031-0.156 0.063-0.188 0.063-0.063 0.125-0.094 0.219-0.094h0.563c0.094 0 0.156 0.031 0.219 0.094 0.031 0.031 0.063 0.125 0.063 0.188zM10.844 6.563v5.156c0 0.094 0 0.156-0.063 0.188-0.063 0.063-0.125 0.094-0.219 0.094h-0.563c-0.094 0-0.156-0.031-0.219-0.094-0.031-0.031-0.063-0.094-0.063-0.188v-5.156c0-0.063 0.031-0.156 0.063-0.188 0.063-0.063 0.125-0.094 0.219-0.094h0.563c0.094 0 0.156 0.031 0.219 0.094 0.063 0.031 0.063 0.125 0.063 0.188zM12 13.031v-8.469h-8v8.469c0 0.125 0.031 0.25 0.063 0.375 0.031 0.094 0.094 0.188 0.125 0.219 0.063 0.063 0.094 0.094 0.094 0.094h7.438c0 0 0.031-0.031 0.094-0.094 0.031-0.031 0.094-0.125 0.125-0.219 0.031-0.125 0.063-0.25 0.063-0.375zM6 3.438h4l-0.438-1.063c-0.031-0.031-0.094-0.063-0.156-0.094h-2.813c-0.063 0.031-0.125 0.063-0.156 0.094zM14.281 3.719v0.563c0 0.094-0.031 0.156-0.063 0.219-0.063 0.031-0.125 0.063-0.219 0.063h-0.844v8.469c0 0.5-0.156 0.938-0.438 1.281-0.281 0.375-0.625 0.531-1 0.531h-7.438c-0.375 0-0.719-0.156-1-0.5s-0.438-0.781-0.438-1.281v-8.5h-0.844c-0.094 0-0.156-0.031-0.219-0.063-0.031-0.063-0.063-0.125-0.063-0.219v-0.563c0-0.094 0.031-0.156 0.063-0.219 0.063-0.031 0.125-0.063 0.219-0.063h2.75l0.625-1.5c0.094-0.219 0.25-0.406 0.5-0.563 0.219-0.156 0.469-0.219 0.688-0.219h2.875c0.219 0 0.469 0.063 0.688 0.219 0.25 0.156 0.406 0.344 0.5 0.563l0.625 1.5h2.75c0.094 0 0.156 0.031 0.219 0.063 0.031 0.063 0.063 0.125 0.063 0.219z"></path></symbol><symbol id="icon-bold" viewBox="0 0 16 16"><path d="M6.74 13.244c0.439 0.176 0.82 0.264 1.172 0.264 0.732 0 1.348-0.117 1.816-0.352s0.82-0.557 1.025-0.938c0.205-0.41 0.322-0.908 0.322-1.523 0-0.645-0.117-1.143-0.352-1.494-0.322-0.527-0.703-0.879-1.172-1.055-0.439-0.205-1.143-0.293-2.080-0.293-0.41 0-0.674 0.029-0.85 0.088v2.666l0.029 2.256c0 0.088 0.029 0.205 0.088 0.381zM6.623 6.975c0.264 0.059 0.557 0.059 0.938 0.059 0.967 0 1.699-0.176 2.197-0.527 0.498-0.381 0.732-0.996 0.732-1.875 0-0.615-0.234-1.143-0.703-1.582-0.469-0.41-1.172-0.615-2.139-0.615-0.293 0-0.645 0.029-1.084 0.117 0 0.234 0 0.439 0.029 0.645 0.029 0.674 0.059 1.465 0.029 2.314v0.82c0 0.264 0 0.469 0 0.645zM2.111 14.416v-0.762c0.264-0.059 0.439-0.088 0.586-0.117 0.439-0.059 0.762-0.146 1.025-0.264 0.088-0.146 0.146-0.293 0.176-0.41 0.059-0.381 0.088-0.908 0.088-1.641l-0.029-4.16c-0.029-1.406-0.059-2.549-0.088-3.369 0-0.498-0.029-0.791-0.088-0.908 0-0.029-0.029-0.059-0.088-0.117-0.117-0.059-0.293-0.088-0.586-0.117-0.176 0-0.469-0.059-0.938-0.117l-0.059-0.674 2.197-0.059 3.164-0.117h0.381c0.029 0 0.059 0 0.117 0s0.088 0 0.117 0c0 0 0.059 0 0.176 0s0.234 0 0.352 0h0.615c0.498 0 1.025 0.059 1.611 0.234 0.234 0.059 0.498 0.176 0.791 0.322 0.322 0.146 0.615 0.352 0.85 0.615 0.264 0.264 0.439 0.557 0.557 0.879s0.176 0.674 0.176 1.025c0 0.381-0.088 0.762-0.264 1.084-0.205 0.322-0.469 0.615-0.82 0.879-0.146 0.088-0.557 0.322-1.23 0.645 0.967 0.205 1.729 0.615 2.227 1.201s0.762 1.26 0.762 1.992c0 0.41-0.088 0.879-0.234 1.348-0.117 0.352-0.322 0.674-0.586 0.967-0.381 0.41-0.762 0.703-1.172 0.908s-0.996 0.381-1.699 0.498c-0.469 0.088-1.025 0.117-1.67 0.088l-1.641-0.029c-0.469 0-1.318 0.029-2.49 0.088-0.205 0.029-0.967 0.059-2.285 0.088z"></path></symbol><symbol id="icon-chain-broken" viewBox="0 0 15 16"><path d="M3.92 11.348l-2.286 2.286q-0.089 0.080-0.205 0.080-0.107 0-0.205-0.080-0.080-0.089-0.080-0.205t0.080-0.205l2.286-2.286q0.089-0.080 0.205-0.080t0.205 0.080q0.080 0.089 0.080 0.205t-0.080 0.205zM5.429 11.714v2.857q0 0.125-0.080 0.205t-0.205 0.080-0.205-0.080-0.080-0.205v-2.857q0-0.125 0.080-0.205t0.205-0.080 0.205 0.080 0.080 0.205zM3.429 9.714q0 0.125-0.080 0.205t-0.205 0.080h-2.857q-0.125 0-0.205-0.080t-0.080-0.205 0.080-0.205 0.205-0.080h2.857q0.125 0 0.205 0.080t0.080 0.205zM14.714 10.857q0 1.071-0.759 1.813l-1.313 1.304q-0.741 0.741-1.813 0.741-1.080 0-1.821-0.759l-2.982-2.991q-0.188-0.188-0.375-0.5l2.134-0.161 2.438 2.446q0.241 0.241 0.607 0.246t0.607-0.237l1.313-1.304q0.25-0.25 0.25-0.598 0-0.357-0.25-0.607l-2.446-2.455 0.161-2.134q0.313 0.188 0.5 0.375l3 3q0.75 0.768 0.75 1.821zM9.205 4.393l-2.134 0.161-2.438-2.446q-0.25-0.25-0.607-0.25-0.348 0-0.607 0.241l-1.313 1.304q-0.25 0.25-0.25 0.598 0 0.357 0.25 0.607l2.446 2.446-0.161 2.143q-0.313-0.188-0.5-0.375l-3-3q-0.75-0.768-0.75-1.821 0-1.071 0.759-1.813l1.313-1.304q0.741-0.741 1.813-0.741 1.080 0 1.821 0.759l2.982 2.991q0.188 0.188 0.375 0.5zM14.857 5.143q0 0.125-0.080 0.205t-0.205 0.080h-2.857q-0.125 0-0.205-0.080t-0.080-0.205 0.080-0.205 0.205-0.080h2.857q0.125 0 0.205 0.080t0.080 0.205zM10 0.286v2.857q0 0.125-0.080 0.205t-0.205 0.080-0.205-0.080-0.080-0.205v-2.857q0-0.125 0.080-0.205t0.205-0.080 0.205 0.080 0.080 0.205zM13.634 1.634l-2.286 2.286q-0.098 0.080-0.205 0.080t-0.205-0.080q-0.080-0.089-0.080-0.205t0.080-0.205l2.286-2.286q0.089-0.080 0.205-0.080t0.205 0.080q0.080 0.089 0.080 0.205t-0.080 0.205z"></path></symbol><symbol id="icon-check" viewBox="0 0 512 512"><path d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="M6.035 7.195a.644.644 0 00.902 0l5.333-5.304c.246-.274.246-.684 0-.93l-.63-.629c-.245-.246-.656-.246-.93 0L6.5 4.543 2.262.332c-.274-.246-.684-.246-.93 0L.703.961c-.246.246-.246.656 0 .93l5.332 5.304z"></path></symbol><symbol id="icon-clipboard-list" viewBox="0 0 384 512"><path d="M336 64h-80c0-35.3-28.7-64-64-64s-64 28.7-64 64H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h288c26.5 0 48-21.5 48-48V112c0-26.5-21.5-48-48-48zM96 424c-13.3 0-24-10.7-24-24s10.7-24 24-24 24 10.7 24 24-10.7 24-24 24zm0-96c-13.3 0-24-10.7-24-24s10.7-24 24-24 24 10.7 24 24-10.7 24-24 24zm0-96c-13.3 0-24-10.7-24-24s10.7-24 24-24 24 10.7 24 24-10.7 24-24 24zm96-192c13.3 0 24 10.7 24 24s-10.7 24-24 24-24-10.7-24-24 10.7-24 24-24zm128 368c0 4.4-3.6 8-8 8H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16zm0-96c0 4.4-3.6 8-8 8H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16zm0-96c0 4.4-3.6 8-8 8H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16z"></path></symbol><symbol id="icon-code" viewBox="0 0 20 16"><path d="M13 11.5l1.5 1.5 5-5-5-5-1.5 1.5 3.5 3.5z"></path><path d="M7 4.5l-1.5-1.5-5 5 5 5 1.5-1.5-3.5-3.5z"></path><path d="M10.958 2.352l1.085 0.296-3 11-1.085-0.296 3-11z"></path></symbol><symbol id="icon-cog" viewBox="0 0 16 16"><path d="M10.281 8c0-0.625-0.219-1.156-0.656-1.625-0.469-0.438-1-0.656-1.625-0.656s-1.156 0.219-1.625 0.656c-0.438 0.469-0.656 1-0.656 1.625s0.219 1.156 0.656 1.625c0.469 0.438 1 0.656 1.625 0.656s1.156-0.219 1.625-0.656c0.438-0.469 0.656-1 0.656-1.625zM14.844 7.031v1.969c0 0.094 0 0.156-0.063 0.219-0.031 0.063-0.094 0.094-0.188 0.125l-1.625 0.25c-0.125 0.313-0.25 0.594-0.375 0.813 0.219 0.281 0.531 0.688 0.969 1.219 0.063 0.063 0.094 0.156 0.094 0.219 0 0.094-0.031 0.156-0.094 0.219-0.156 0.219-0.438 0.531-0.875 0.969-0.438 0.406-0.719 0.625-0.844 0.625-0.063 0-0.156-0.031-0.219-0.094l-1.25-0.969c-0.25 0.156-0.531 0.25-0.813 0.344-0.094 0.813-0.188 1.375-0.25 1.656-0.031 0.188-0.156 0.25-0.313 0.25h-2c-0.063 0-0.156 0-0.219-0.063s-0.094-0.125-0.094-0.188l-0.25-1.656c-0.281-0.094-0.563-0.188-0.813-0.313l-1.25 0.938c-0.063 0.063-0.125 0.094-0.219 0.094s-0.156-0.031-0.219-0.094c-0.75-0.688-1.25-1.188-1.469-1.5-0.063-0.063-0.063-0.125-0.063-0.219 0-0.063 0-0.125 0.063-0.188 0.094-0.125 0.25-0.344 0.438-0.594 0.219-0.281 0.375-0.5 0.5-0.656-0.156-0.281-0.281-0.594-0.375-0.875l-1.625-0.25c-0.094 0-0.156-0.031-0.188-0.094-0.063-0.063-0.063-0.125-0.063-0.219v-1.969c0-0.094 0-0.156 0.063-0.219 0.031-0.063 0.094-0.094 0.156-0.125l1.656-0.25c0.094-0.25 0.219-0.531 0.375-0.813-0.25-0.344-0.563-0.75-0.969-1.219-0.063-0.094-0.094-0.156-0.094-0.219s0.031-0.125 0.094-0.219c0.156-0.219 0.438-0.531 0.875-0.938 0.438-0.438 0.719-0.656 0.844-0.656 0.063 0 0.156 0.031 0.219 0.094l1.25 0.969c0.25-0.156 0.531-0.25 0.813-0.344 0.094-0.813 0.188-1.375 0.25-1.656 0.031-0.188 0.156-0.25 0.313-0.25h2c0.063 0 0.156 0 0.219 0.063s0.094 0.125 0.094 0.188l0.25 1.656c0.281 0.094 0.563 0.188 0.813 0.313l1.25-0.938c0.063-0.063 0.125-0.094 0.219-0.094s0.156 0.031 0.219 0.094c0.781 0.719 1.281 1.219 1.469 1.531 0.063 0.031 0.063 0.094 0.063 0.188 0 0.063 0 0.125-0.063 0.188-0.094 0.125-0.25 0.344-0.438 0.594-0.219 0.281-0.375 0.5-0.5 0.656 0.156 0.281 0.281 0.563 0.375 0.875l1.625 0.25c0.094 0 0.156 0.031 0.188 0.094 0.063 0.063 0.063 0.125 0.063 0.219z"></path></symbol><symbol id="icon-cogs" viewBox="0 0 16 16"><path d="M7.469 8c0-0.594-0.219-1.094-0.625-1.5-0.406-0.438-0.906-0.625-1.5-0.625s-1.094 0.188-1.531 0.625c-0.406 0.406-0.625 0.906-0.625 1.5s0.219 1.094 0.625 1.5c0.438 0.438 0.938 0.625 1.531 0.625s1.094-0.188 1.5-0.625c0.406-0.406 0.625-0.906 0.625-1.5zM13.875 12.281c0-0.313-0.125-0.563-0.313-0.75-0.219-0.219-0.469-0.344-0.75-0.344-0.313 0-0.563 0.125-0.75 0.344-0.219 0.188-0.344 0.438-0.344 0.75 0 0.281 0.125 0.531 0.313 0.75 0.219 0.188 0.469 0.313 0.781 0.313 0.281 0 0.531-0.125 0.75-0.313 0.188-0.219 0.313-0.469 0.313-0.75zM13.875 3.719c0-0.281-0.125-0.531-0.313-0.75-0.219-0.188-0.469-0.313-0.75-0.313-0.313 0-0.563 0.125-0.75 0.313-0.219 0.219-0.344 0.469-0.344 0.75 0 0.313 0.125 0.563 0.313 0.781 0.219 0.188 0.469 0.313 0.781 0.313 0.281 0 0.531-0.125 0.75-0.313 0.188-0.219 0.313-0.469 0.313-0.781zM10.656 7.25v1.531c0 0.063 0 0.125-0.063 0.156-0.031 0.063-0.063 0.094-0.125 0.094l-1.281 0.188c-0.063 0.219-0.156 0.406-0.281 0.656 0.188 0.25 0.438 0.563 0.75 0.938 0.063 0.063 0.063 0.125 0.063 0.188s0 0.125-0.063 0.156c-0.125 0.156-0.344 0.406-0.688 0.75-0.313 0.313-0.531 0.5-0.656 0.5-0.063 0-0.094-0.031-0.156-0.063l-0.969-0.75c-0.188 0.094-0.406 0.188-0.625 0.25-0.063 0.594-0.125 1.031-0.219 1.281-0.031 0.156-0.094 0.219-0.25 0.219h-1.531c-0.063 0-0.125-0.031-0.156-0.063-0.063-0.063-0.094-0.094-0.094-0.156l-0.188-1.281c-0.188-0.063-0.406-0.125-0.625-0.25l-1 0.75c-0.031 0.031-0.094 0.063-0.156 0.063s-0.125-0.031-0.188-0.094c-0.781-0.719-1.188-1.156-1.188-1.313 0-0.063 0.031-0.125 0.063-0.156 0.063-0.094 0.156-0.25 0.344-0.438 0.156-0.219 0.281-0.406 0.375-0.531-0.125-0.25-0.219-0.469-0.281-0.688l-1.281-0.188c-0.031 0-0.094-0.031-0.125-0.094-0.031-0.031-0.063-0.094-0.063-0.156v-1.531c0-0.063 0.031-0.125 0.063-0.156 0.031-0.063 0.094-0.094 0.125-0.094l1.281-0.188c0.063-0.219 0.156-0.406 0.281-0.656-0.188-0.25-0.438-0.563-0.75-0.938-0.031-0.063-0.063-0.125-0.063-0.188s0.031-0.125 0.063-0.156c0.125-0.156 0.344-0.406 0.688-0.75 0.344-0.313 0.563-0.5 0.656-0.5 0.063 0 0.125 0.031 0.188 0.063l0.938 0.75c0.188-0.094 0.406-0.188 0.656-0.25 0.063-0.594 0.125-1.031 0.188-1.281 0.031-0.156 0.125-0.219 0.25-0.219h1.531c0.063 0 0.125 0.031 0.188 0.063 0.031 0.063 0.063 0.094 0.063 0.156l0.219 1.281c0.188 0.063 0.375 0.125 0.625 0.25l0.969-0.75c0.031-0.031 0.094-0.063 0.156-0.063s0.125 0.031 0.188 0.094c0.813 0.719 1.188 1.156 1.188 1.313 0 0.063 0 0.125-0.031 0.156-0.094 0.094-0.188 0.25-0.375 0.469-0.156 0.188-0.281 0.375-0.375 0.5 0.125 0.25 0.219 0.5 0.281 0.688l1.281 0.188c0.063 0 0.094 0.031 0.125 0.094 0.063 0.031 0.063 0.094 0.063 0.156zM16 11.688v1.156c0 0.094-0.406 0.188-1.25 0.25-0.063 0.156-0.156 0.313-0.25 0.438 0.281 0.625 0.438 1.031 0.438 1.156 0 0.031 0 0.031-0.031 0.063-0.688 0.406-1.031 0.594-1.031 0.594-0.063 0-0.188-0.125-0.406-0.406-0.188-0.25-0.344-0.438-0.406-0.563-0.125 0.031-0.219 0.031-0.25 0.031-0.063 0-0.156 0-0.25-0.031-0.094 0.125-0.25 0.313-0.438 0.563-0.219 0.281-0.344 0.406-0.406 0.406 0 0-0.344-0.188-1.031-0.594 0-0.031-0.031-0.031-0.031-0.063 0-0.125 0.156-0.531 0.438-1.156-0.094-0.125-0.188-0.281-0.25-0.438-0.844-0.063-1.25-0.156-1.25-0.25v-1.156c0-0.094 0.406-0.188 1.25-0.25 0.063-0.188 0.156-0.313 0.25-0.438-0.281-0.625-0.438-1.031-0.438-1.156 0-0.031 0.031-0.031 0.031-0.063 0.031 0 0.125-0.063 0.313-0.156 0.156-0.094 0.313-0.188 0.469-0.281s0.25-0.156 0.25-0.156c0.063 0 0.188 0.156 0.406 0.406 0.188 0.25 0.344 0.438 0.438 0.563 0.094-0.031 0.188-0.031 0.25-0.031 0.031 0 0.125 0 0.25 0.031 0.281-0.406 0.531-0.719 0.75-0.938l0.063-0.031c0 0 0.344 0.219 1.031 0.594 0.031 0.031 0.031 0.031 0.031 0.063 0 0.125-0.156 0.531-0.438 1.156 0.094 0.125 0.188 0.25 0.25 0.438 0.844 0.063 1.25 0.156 1.25 0.25zM16 3.156v1.156c0 0.094-0.406 0.188-1.25 0.25-0.063 0.156-0.156 0.313-0.25 0.438 0.281 0.625 0.438 1.031 0.438 1.156 0 0.031 0 0.031-0.031 0.063-0.688 0.406-1.031 0.594-1.031 0.594-0.063 0-0.188-0.125-0.406-0.406-0.188-0.25-0.344-0.438-0.406-0.563-0.125 0.031-0.219 0.031-0.25 0.031-0.063 0-0.156 0-0.25-0.031-0.094 0.125-0.25 0.313-0.438 0.563-0.219 0.281-0.344 0.406-0.406 0.406 0 0-0.344-0.188-1.031-0.594 0-0.031-0.031-0.031-0.031-0.063 0-0.125 0.156-0.531 0.438-1.156-0.094-0.125-0.188-0.281-0.25-0.438-0.844-0.063-1.25-0.156-1.25-0.25v-1.156c0-0.094 0.406-0.188 1.25-0.25 0.063-0.188 0.156-0.313 0.25-0.438-0.281-0.625-0.438-1.031-0.438-1.156 0-0.031 0.031-0.031 0.031-0.063 0.031 0 0.125-0.063 0.313-0.156 0.156-0.125 0.313-0.219 0.469-0.281 0.156-0.094 0.25-0.156 0.25-0.156 0.063 0 0.188 0.125 0.406 0.406 0.188 0.25 0.344 0.438 0.438 0.563 0.094-0.031 0.188-0.031 0.25-0.031 0.031 0 0.125 0 0.25 0.031 0.281-0.406 0.531-0.719 0.75-0.938l0.063-0.031c0 0 0.344 0.219 1.031 0.594 0.031 0.031 0.031 0.031 0.031 0.063 0 0.125-0.156 0.531-0.438 1.156 0.094 0.125 0.188 0.25 0.25 0.438 0.844 0.063 1.25 0.156 1.25 0.25z"></path></symbol><symbol id="icon-collapse-down" viewBox="0 0 16 16"><path d="M4.219 1.75h7.531v0.719h-7.531zM12.656 7.281l-3.969 3.875v-6.938h-1.406v6.875l-3.906-3.813-1.031 1 5.656 5.656 5.656-5.656z"></path></symbol><symbol id="icon-collapse-up" viewBox="0 0 16 16"><path d="M4.25 1.75h7.531v0.719h-7.531zM3.375 11.188l3.938-3.875v6.938h1.406v-6.875l3.938 3.813 1-1-5.656-5.656-5.656 5.656z"></path></symbol><symbol id="icon-comment-add-reversed" viewBox="0 0 30 30"><path d="M15 8c-4.438 0-8 2.938-8 6.5 0 1.563.656 2.969 1.781 4.094-.406 1.593-1.719 2.968-1.719 3a.3.3 0 00-.062.281c.063.094.125.125.25.125 2.063 0 3.625-.969 4.375-1.594A9.81 9.81 0 0015 21c4.406 0 8-2.906 8-6.5 0-3.563-3.594-6.5-8-6.5z" fill="currentColor"/><path d="M17.625 13.719h-1.969V11.75a.45.45 0 00-.437-.438h-.438a.44.44 0 00-.437.438v1.969h-1.969a.44.44 0 00-.438.437v.438c0 .246.192.437.438.437h1.969V17c0 .246.191.438.437.438h.438a.44.44 0 00.437-.438v-1.969h1.969a.44.44 0 00.438-.437v-.438a.45.45 0 00-.438-.437z" fill="#fff"></path></symbol><symbol id="icon-comment-add" viewBox="0 0 30 30"><path d="M15 8c-4.438 0-8 2.938-8 6.5 0 1.5.594 2.875 1.625 3.969-.438 1.218-1.406 2.281-1.438 2.281-.218.219-.25.531-.156.813.125.28.407.437.719.437 1.906 0 3.438-.781 4.344-1.438A9.753 9.753 0 0015 21c4.406 0 8-2.906 8-6.5 0-3.563-3.594-6.5-8-6.5zm0 11.5a8.15 8.15 0 01-2.469-.375l-.719-.219-.593.438a6.979 6.979 0 01-1.813.906c.25-.375.469-.813.625-1.25l.344-.875-.656-.688C9.156 16.845 8.5 15.845 8.5 14.5c0-2.75 2.906-5 6.5-5 3.563 0 6.5 2.25 6.5 5 0 2.781-2.938 5-6.5 5z" fill="currentColor"></path><path d="M17.625 13.719h-1.969V11.75a.45.45 0 00-.437-.438h-.438a.44.44 0 00-.437.438v1.969h-1.969a.44.44 0 00-.438.437v.438c0 .246.192.437.438.437h1.969V17c0 .246.191.438.437.438h.438a.44.44 0 00.437-.438v-1.969h1.969a.44.44 0 00.438-.437v-.438a.45.45 0 00-.438-.437z" fill="currentColor"></path></symbol><symbol id="icon-comment-large-outline" viewBox="0 0 17 17"><path fill-rule="evenodd" clip-rule="evenodd" d="M1 6.5C1 2.937 4.563 0 9 0c4.406 0 8 2.938 8 6.5 0 3.594-3.594 6.5-8 6.5-1.031 0-2-.156-2.906-.438C5.188 13.22 3.656 14 1.75 14c-.312 0-.594-.156-.719-.438-.093-.28-.062-.593.157-.812.03 0 1-1.063 1.437-2.281C1.594 9.375 1 8 1 6.5zm5.531 4.625A8.15 8.15 0 009 11.5c3.563 0 6.5-2.219 6.5-5 0-2.75-2.937-5-6.5-5-3.594 0-6.5 2.25-6.5 5 0 1.344.656 2.344 1.219 2.938l.656.687-.344.875a5.844 5.844 0 01-.625 1.25 6.976 6.976 0 001.813-.906l.594-.438.718.219zm3.125-5.406h1.969a.45.45 0 01.438.437v.438a.44.44 0 01-.438.437H9.656V9a.44.44 0 01-.437.438H8.78A.432.432 0 018.344 9V7.031H6.375a.432.432 0 01-.437-.437v-.438a.44.44 0 01.437-.437h1.969V3.75a.44.44 0 01.437-.438h.438a.45.45 0 01.437.438v1.969z" fill="#007D7E"></path></symbol><symbol id="icon-comment-large-reversed" viewBox="0 0 17 17"><path fill-rule="evenodd" clip-rule="evenodd" d="M1 6.5C1 2.937 4.563 0 9 0c4.406 0 8 2.938 8 6.5 0 3.594-3.594 6.5-8 6.5a9.81 9.81 0 01-3.375-.594C4.875 13.031 3.313 14 1.25 14c-.125 0-.187-.031-.25-.125a.3.3 0 01.063-.281.83.83 0 01.074-.094c.284-.337 1.296-1.541 1.644-2.906C1.656 9.469 1 8.063 1 6.5zm8.656-.781h1.969a.45.45 0 01.438.437v.438a.44.44 0 01-.438.437H9.656V9a.44.44 0 01-.437.438H8.78A.432.432 0 018.344 9V7.031H6.375a.432.432 0 01-.438-.437v-.438a.44.44 0 01.438-.437h1.969V3.75a.44.44 0 01.437-.438h.438a.45.45 0 01.437.438v1.969z" fill="#007D7E"></path></symbol><symbol id="icon-comment-large" viewBox="0 0 17 14"><path d="M9 0C4.563 0 1 2.938 1 6.5c0 1.563.656 2.969 1.781 4.094-.406 1.594-1.718 2.969-1.718 3a.3.3 0 00-.063.281c.063.094.125.125.25.125 2.063 0 3.625-.969 4.375-1.594A9.81 9.81 0 009 13c4.406 0 8-2.906 8-6.5C17 2.937 13.406 0 9 0zM5 7.5c-.562 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1zm4 0c-.562 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1zm4 0c-.562 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1z" fill="#007D7E"></path></symbol><symbol id="icon-comment" viewBox="0 0 30 30"><path d="M15 8c-4.438 0-8 2.938-8 6.5 0 1.563.656 2.969 1.781 4.094-.406 1.593-1.719 2.968-1.719 3a.3.3 0 00-.062.281c.063.094.125.125.25.125 2.063 0 3.625-.969 4.375-1.594A9.81 9.81 0 0015 21c4.406 0 8-2.906 8-6.5 0-3.563-3.594-6.5-8-6.5zm-4 7.5c-.563 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1zm4 0c-.563 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1zm4 0c-.563 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1z" fill="currentColor"></path></symbol><symbol id="icon-cross" viewBox="0 0 16 16"><path d="M13.313 11.313c0 0.219-0.094 0.438-0.25 0.594l-1.219 1.219c-0.156 0.156-0.375 0.25-0.625 0.25-0.219 0-0.438-0.094-0.594-0.25l-2.625-2.625-2.625 2.625c-0.156 0.156-0.375 0.25-0.594 0.25-0.25 0-0.469-0.094-0.625-0.25l-1.219-1.219c-0.156-0.156-0.25-0.375-0.25-0.594 0-0.25 0.094-0.438 0.25-0.625l2.625-2.625-2.625-2.625c-0.156-0.156-0.25-0.375-0.25-0.594 0-0.25 0.094-0.438 0.25-0.625l1.219-1.188c0.156-0.188 0.375-0.25 0.625-0.25 0.219 0 0.438 0.063 0.594 0.25l2.625 2.625 2.625-2.625c0.156-0.188 0.375-0.25 0.594-0.25 0.25 0 0.469 0.063 0.625 0.25l1.219 1.188c0.156 0.188 0.25 0.375 0.25 0.625 0 0.219-0.094 0.438-0.25 0.594l-2.625 2.625 2.625 2.625c0.156 0.188 0.25 0.375 0.25 0.625z"></path></symbol><symbol id="icon-date" viewBox="0 0 16 16"><path d="M12.12 14.592h-8.24c-1.365 0-2.472-1.107-2.472-2.472v-8.24c0-1.081 0.695-1.983 1.648-2.317v1.081c0 0.695 0.566 1.236 1.236 1.236s1.236-0.541 1.236-1.236v-1.236h4.944v1.236c0 0.695 0.566 1.236 1.236 1.236s1.236-0.541 1.236-1.236v-1.081c0.953 0.335 1.648 1.236 1.648 2.317v8.24c0 1.365-1.107 2.472-2.472 2.472zM12.944 5.528h-9.888v6.592c0 0.463 0.36 0.824 0.824 0.824h8.24c0.463 0 0.824-0.36 0.824-0.824zM9.648 9.648h1.648v1.648h-1.648zM9.648 7.176h1.648v1.648h-1.648zM7.176 9.648h1.648v1.648h-1.648zM7.176 7.176h1.648v1.648h-1.648zM4.704 9.648h1.648v1.648h-1.648zM4.704 7.176h1.648v1.648h-1.648zM11.708 3.056c-0.232 0-0.412-0.18-0.412-0.412v-1.236h0.824v1.236c0 0.232-0.18 0.412-0.412 0.412zM4.292 3.056c-0.232 0-0.412-0.18-0.412-0.412v-1.236h0.824v1.236c0 0.232-0.18 0.412-0.412 0.412z"></path></symbol><symbol id="icon-doc-empty-inverse" viewBox="0 0 16 16"><path d="M14.1 6.767v7.633c0 0.267-0.1 0.467-0.267 0.633-0.167 0.2-0.4 0.267-0.667 0.267h-10.333c-0.267 0-0.5-0.067-0.667-0.267-0.167-0.167-0.267-0.367-0.267-0.633v-12.8c0-0.267 0.1-0.467 0.267-0.633 0.167-0.2 0.4-0.267 0.667-0.267h5.167v5.167c0 0.267 0.1 0.467 0.267 0.633 0.167 0.2 0.4 0.267 0.633 0.267zM14.067 5.567h-4.833v-4.867c0.5 0.1 0.933 0.3 1.233 0.633l2.967 2.967c0.333 0.333 0.533 0.733 0.633 1.267z"></path></symbol><symbol id="icon-doc-empty" viewBox="0 0 16 16"><path d="M3.133 14.1h9.733v-7.333h-3.967c-0.233 0-0.467-0.067-0.633-0.267-0.167-0.167-0.267-0.367-0.267-0.633v-3.967h-4.867zM9.233 5.567h3.567c-0.067-0.2-0.133-0.333-0.2-0.4l-3-2.967c-0.067-0.1-0.2-0.167-0.367-0.233zM14.1 5.867v8.533c0 0.267-0.1 0.467-0.267 0.633-0.167 0.2-0.4 0.267-0.667 0.267h-10.333c-0.267 0-0.5-0.067-0.667-0.267-0.167-0.167-0.267-0.367-0.267-0.633v-12.8c0-0.267 0.1-0.467 0.267-0.633 0.167-0.2 0.4-0.267 0.667-0.267h6.067c0.267 0 0.533 0.033 0.867 0.167 0.3 0.133 0.533 0.3 0.7 0.467l2.967 2.967c0.2 0.167 0.333 0.433 0.467 0.733s0.2 0.567 0.2 0.833z"></path></symbol><symbol id="icon-doc-full-inverse" viewBox="0 0 16 16"><path d="M11.438 12.281v-0.563c0-0.094-0.031-0.156-0.094-0.219-0.063-0.031-0.125-0.063-0.188-0.063h-6.313c-0.063 0-0.125 0.031-0.188 0.063-0.063 0.063-0.094 0.125-0.094 0.219v0.563c0 0.094 0.031 0.156 0.094 0.219 0.063 0.031 0.125 0.063 0.188 0.063h6.313c0.063 0 0.125-0.031 0.188-0.063 0.063-0.063 0.094-0.125 0.094-0.219zM11.438 10v-0.563c0-0.094-0.031-0.156-0.094-0.219s-0.125-0.063-0.188-0.063h-6.313c-0.063 0-0.125 0-0.188 0.063s-0.094 0.125-0.094 0.219v0.563c0 0.094 0.031 0.156 0.094 0.219 0.063 0.031 0.125 0.063 0.188 0.063h6.313c0.063 0 0.125-0.031 0.188-0.063 0.063-0.063 0.094-0.125 0.094-0.219zM13.719 6.844v7.156c0 0.25-0.094 0.438-0.25 0.594-0.156 0.188-0.375 0.25-0.625 0.25h-9.688c-0.25 0-0.469-0.063-0.625-0.25-0.156-0.156-0.25-0.344-0.25-0.594v-12c0-0.25 0.094-0.438 0.25-0.594 0.156-0.188 0.375-0.25 0.625-0.25h4.844v4.844c0 0.25 0.094 0.438 0.25 0.594 0.156 0.188 0.375 0.25 0.594 0.25zM13.688 5.719h-4.531v-4.563c0.469 0.094 0.875 0.281 1.156 0.594l2.781 2.781c0.313 0.313 0.5 0.688 0.594 1.188z"></path></symbol><symbol id="icon-doc-full" viewBox="0 0 16 16"><path d="M11.438 10.563v0.594c0 0.063-0.031 0.125-0.094 0.188s-0.125 0.094-0.188 0.094h-6.313c-0.063 0-0.125-0.031-0.188-0.094s-0.094-0.125-0.094-0.188v-0.594c0-0.063 0.031-0.156 0.094-0.188 0.063-0.063 0.125-0.094 0.188-0.094h6.313c0.063 0 0.125 0.031 0.188 0.094 0.063 0.031 0.094 0.125 0.094 0.188zM11.438 8.281v0.563c0 0.094-0.031 0.156-0.094 0.219s-0.125 0.094-0.188 0.094h-6.313c-0.063 0-0.125-0.031-0.188-0.094s-0.094-0.125-0.094-0.219v-0.563c0-0.094 0.031-0.156 0.094-0.188 0.063-0.063 0.125-0.094 0.188-0.094h6.313c0.063 0 0.125 0.031 0.188 0.094 0.063 0.031 0.094 0.094 0.094 0.188zM3.438 13.719h9.125v-6.875h-3.719c-0.219 0-0.438-0.063-0.594-0.25-0.156-0.156-0.25-0.344-0.25-0.594v-3.719h-4.563zM9.156 5.719h3.344c-0.063-0.188-0.125-0.313-0.188-0.375l-2.813-2.781c-0.063-0.094-0.188-0.156-0.344-0.219zM13.719 6v8c0 0.25-0.094 0.438-0.25 0.594-0.156 0.188-0.375 0.25-0.625 0.25h-9.688c-0.25 0-0.469-0.063-0.625-0.25-0.156-0.156-0.25-0.344-0.25-0.594v-12c0-0.25 0.094-0.438 0.25-0.594 0.156-0.188 0.375-0.25 0.625-0.25h5.688c0.25 0 0.5 0.031 0.813 0.156 0.281 0.125 0.5 0.281 0.656 0.438l2.781 2.781c0.188 0.156 0.313 0.406 0.438 0.688s0.188 0.531 0.188 0.781z"></path></symbol><symbol id="icon-download-alt" viewBox="0 0 512 512"><path d="M216 0h80c13.3 0 24 10.7 24 24v168h87.7c17.8 0 26.7 21.5 14.1 34.1L269.7 378.3c-7.5 7.5-19.8 7.5-27.3 0L90.1 226.1c-12.6-12.6-3.7-34.1 14.1-34.1H192V24c0-13.3 10.7-24 24-24zm296 376v112c0 13.3-10.7 24-24 24H24c-13.3 0-24-10.7-24-24V376c0-13.3 10.7-24 24-24h146.7l49 49c20.1 20.1 52.5 20.1 72.6 0l49-49H488c13.3 0 24 10.7 24 24zm-124 88c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20zm64 0c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20z"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="M12.609 9.802v3.013h-9.218v-3.013h-1.751v3.888c0 0.489 0.386 0.901 0.875 0.901h10.969c0.489 0 0.875-0.412 0.875-0.901v-3.888zM7.846 9.493l-2.549-3.038c0 0-0.386-0.386 0.051-0.386 0.412 0 1.416 0 1.416 0s0-0.232 0-0.618c0-1.056 0-3.038 0-3.837 0 0-0.051-0.206 0.283-0.206 0.309 0 1.751 0 2.008 0 0.232 0 0.206 0.18 0.206 0.18 0 0.772 0 2.807 0 3.837 0 0.335 0 0.566 0 0.566s0.824 0 1.339 0c0.489 0 0.129 0.386 0.129 0.386s-2.163 2.858-2.472 3.141c-0.206 0.232-0.412-0.026-0.412-0.026z"></path></symbol><symbol id="icon-draft" viewBox="0 0 23 21"><g fill-rule="evenodd"><path d="M18 18v1.11a.88.88 0 01-.875.89H4.875A.88.88 0 014 19.11V18h14zM12.167 1v5.047c0 .49.393.89.875.89H18V9H4V1.89c0-.455.332-.828.765-.883L4.875 1h7.292zm1.071 0c.25 0 .489.098.664.273l3.825 3.829a.935.935 0 01.273.66V6h-5V1h.238z" fill-rule="nonzero"/><text font-family="Open Sans, Arial, sans-serif" font-size="7" font-weight="bold" transform="translate(0 .333)"><tspan x="0" y="15.667">DRAFT</tspan></text></g></symbol><symbol id="icon-duplicate" viewBox="0 0 16 16"><path d="M 5.10985,1 C 4.89773,1 4.71212,1.053 4.57955,1.21212 4.44697,1.3447 4.36742,1.50379 4.36742,1.71591 V 3.38636 H 2.67045 c -0.21211,0 -0.39772,0.053 -0.5303,0.21213 C 2.00758,3.73106 1.92803,3.89015 1.92803,4.10227 v 10.18182 c 0,0.21212 0.0795,0.37121 0.21212,0.50379 C 2.27273,14.94697 2.45834,15 2.67045,15 h 8.2197 c 0.21212,0 0.39773,-0.053 0.5303,-0.21212 0.13258,-0.13258 0.21213,-0.29167 0.21213,-0.50379 v -1.67045 h 1.69697 c 0.21211,0 0.39772,-0.053 0.5303,-0.21213 0.13257,-0.13257 0.21212,-0.29166 0.21212,-0.50378 V 5.10985 c 0,-0.21212 -0.053,-0.42425 -0.15909,-0.66288 C 13.80682,4.20834 13.70076,3.99621 13.54166,3.86364 L 11.18182,1.50379 C 11.04924,1.37121 10.86364,1.23864 10.625,1.13258 10.35985,1.02651 10.14773,1 9.9356,1 Z M 5.34849,1.95455 H 9.2197 v 3.1553 c 0,0.21212 0.0795,0.37121 0.21211,0.50379 C 9.5644,5.77273 9.75,5.82576 9.9356,5.82576 h 3.15531 v 5.83333 H 5.34849 Z m 4.85227,0.053 c 0.13258,0.053 0.23863,0.10606 0.29166,0.18561 l 2.38637,2.35985 c 0.053,0.053 0.10606,0.15908 0.15909,0.31817 H 10.20076 Z M 2.90909,4.34088 h 1.45833 v 7.55682 c 0,0.21212 0.0795,0.37121 0.21213,0.50378 0.13257,0.1591 0.31818,0.21213 0.5303,0.21213 h 5.54166 v 1.43182 H 2.90909 Z" /></symbol><symbol id="icon-edit" viewBox="0 0 16 16"><path d="M12.12 1.008l-0.876 0.94 2.501 2.33 0.876-0.94-2.501-2.33zM10.667 2.568l-8.592 9.234 2.501 2.33 8.592-9.234-2.501-2.33zM1.626 12.55l-0.192 2.458 2.437-0.363-2.244-2.095z"></path></symbol><symbol id="icon-ellipsis-v" viewBox="0 0 192 512"><path d="M96 184c39.8 0 72 32.2 72 72s-32.2 72-72 72-72-32.2-72-72 32.2-72 72-72zM24 80c0 39.8 32.2 72 72 72s72-32.2 72-72S135.8 8 96 8 24 40.2 24 80zm0 352c0 39.8 32.2 72 72 72s72-32.2 72-72-32.2-72-72-72-72 32.2-72 72z"></path></symbol><symbol id="icon-error" viewBox="0 0 512 512"><path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/></symbol><symbol id="icon-folder-inverse" viewBox="0 0 16 16"><path d="M14.973 6.098v5.889c0 0.498-0.205 0.938-0.557 1.318-0.381 0.381-0.82 0.557-1.318 0.557h-10.195c-0.498 0-0.938-0.176-1.318-0.557-0.352-0.381-0.557-0.82-0.557-1.318v-8.027c0-0.527 0.205-0.967 0.557-1.348 0.381-0.352 0.82-0.527 1.318-0.527h2.695c0.498 0 0.938 0.176 1.318 0.527 0.352 0.381 0.557 0.82 0.557 1.348v0.264h5.625c0.498 0 0.938 0.176 1.318 0.557 0.352 0.352 0.557 0.791 0.557 1.318z"></path></symbol><symbol id="icon-folder-open-1" viewBox="0 0 16 16"><path d="M14.015 8.747c0-0.165-0.137-0.247-0.385-0.247h-7.965c-0.192 0-0.412 0.055-0.632 0.165s-0.412 0.22-0.522 0.385l-2.17 2.664c-0.082 0.11-0.11 0.192-0.11 0.275 0 0.165 0.11 0.275 0.385 0.275h7.965c0.192 0 0.412-0.055 0.632-0.165s0.385-0.247 0.522-0.385l2.142-2.664c0.082-0.11 0.137-0.22 0.137-0.302zM5.665 7.566h5.603v-1.181c0-0.192-0.055-0.357-0.192-0.494s-0.302-0.192-0.494-0.192h-4.23c-0.192 0-0.357-0.082-0.494-0.22s-0.192-0.302-0.192-0.494v-0.467c0-0.192-0.082-0.357-0.22-0.494s-0.302-0.22-0.494-0.22h-2.335c-0.192 0-0.357 0.082-0.494 0.22s-0.22 0.302-0.22 0.494v6.235l1.868-2.307c0.22-0.247 0.522-0.467 0.851-0.632 0.357-0.165 0.714-0.247 1.044-0.247zM14.949 8.747c0 0.302-0.11 0.604-0.33 0.879l-2.17 2.664c-0.22 0.275-0.494 0.467-0.851 0.632-0.357 0.192-0.687 0.275-1.016 0.275h-7.965c-0.467 0-0.851-0.165-1.154-0.494-0.33-0.33-0.494-0.714-0.494-1.154v-7.031c0-0.439 0.165-0.824 0.494-1.154 0.302-0.33 0.687-0.494 1.154-0.494h2.335c0.439 0 0.824 0.165 1.154 0.494s0.494 0.714 0.494 1.154v0.22h3.983c0.439 0 0.824 0.165 1.154 0.494s0.494 0.714 0.494 1.154v1.181h1.401c0.247 0 0.494 0.055 0.714 0.165 0.22 0.137 0.385 0.302 0.494 0.522 0.082 0.165 0.11 0.33 0.11 0.494z"></path></symbol><symbol id="icon-folder-open-inverse" viewBox="0 0 16 16"><path d="M14.729 8.912c0 0.137-0.082 0.302-0.22 0.494l-2.472 2.884c-0.192 0.247-0.494 0.467-0.879 0.632-0.385 0.192-0.714 0.275-1.044 0.275h-7.965c-0.165 0-0.33-0.027-0.439-0.11-0.137-0.055-0.22-0.165-0.22-0.302 0-0.165 0.082-0.33 0.247-0.494l2.444-2.884c0.22-0.275 0.522-0.467 0.879-0.659 0.385-0.165 0.742-0.247 1.071-0.247h7.965c0.165 0 0.302 0.027 0.439 0.082 0.137 0.082 0.192 0.192 0.192 0.33zM12.23 6.385v1.181h-6.097c-0.467 0-0.934 0.11-1.456 0.357-0.494 0.22-0.906 0.522-1.208 0.851l-2.444 2.911-0.055 0.055c0-0.027 0-0.055 0-0.11 0-0.027 0-0.055 0-0.082v-7.031c0-0.439 0.165-0.824 0.494-1.154 0.302-0.33 0.687-0.494 1.154-0.494h2.335c0.439 0 0.824 0.165 1.154 0.494s0.494 0.714 0.494 1.154v0.22h3.983c0.439 0 0.824 0.165 1.154 0.494s0.494 0.714 0.494 1.154z"></path></symbol><symbol id="icon-folder" viewBox="0 0 16 16"><path d="M13.889 11.986v-5.889c0-0.234-0.088-0.41-0.234-0.586-0.146-0.146-0.352-0.234-0.557-0.234h-5.889c-0.234 0-0.439-0.059-0.586-0.234-0.146-0.146-0.234-0.352-0.234-0.557v-0.527c0-0.234-0.088-0.439-0.234-0.586s-0.352-0.234-0.557-0.234h-2.695c-0.205 0-0.41 0.088-0.557 0.234s-0.234 0.352-0.234 0.586v8.027c0 0.205 0.088 0.41 0.234 0.557 0.146 0.176 0.352 0.234 0.557 0.234h10.195c0.205 0 0.41-0.059 0.557-0.234 0.146-0.146 0.234-0.352 0.234-0.557zM14.973 6.098v5.889c0 0.498-0.205 0.938-0.557 1.318-0.381 0.381-0.82 0.557-1.318 0.557h-10.195c-0.498 0-0.938-0.176-1.318-0.557-0.352-0.381-0.557-0.82-0.557-1.318v-8.027c0-0.527 0.205-0.967 0.557-1.348 0.381-0.352 0.82-0.527 1.318-0.527h2.695c0.498 0 0.938 0.176 1.318 0.527 0.352 0.381 0.557 0.82 0.557 1.348v0.264h5.625c0.498 0 0.938 0.176 1.318 0.557 0.352 0.352 0.557 0.791 0.557 1.318z"></path></symbol><symbol id="icon-form" viewBox="0 0 16 16"><path d="M1.167 1.16v13.659h13.659v-13.659h-13.659zM5.345 12.661h-2.755v-1.423h5.027v1.423zM12.668 5.476h-10.078v-2.135h10.055v2.135zM12.646 9.333h-10.055v-2.135h10.055v2.135z"></path></symbol><symbol id="icon-grip" viewBox="0 0 16 16"><path d="M7.022 12.97c0 0.901-0.747 1.622-1.648 1.622s-1.648-0.721-1.648-1.622c0-0.927 0.747-1.648 1.648-1.648s1.648 0.721 1.648 1.648zM7.022 8c0 0.901-0.747 1.648-1.648 1.648s-1.648-0.747-1.648-1.648c0-0.901 0.747-1.648 1.648-1.648s1.648 0.747 1.648 1.648zM7.022 3.030c0 0.927-0.747 1.648-1.648 1.648s-1.648-0.721-1.648-1.648c0-0.901 0.747-1.622 1.648-1.622s1.648 0.721 1.648 1.622zM12.274 12.97c0 0.901-0.747 1.622-1.648 1.622s-1.648-0.721-1.648-1.622c0-0.927 0.747-1.648 1.648-1.648s1.648 0.721 1.648 1.648zM12.274 8c0 0.901-0.747 1.648-1.648 1.648s-1.648-0.747-1.648-1.648c0-0.901 0.747-1.648 1.648-1.648s1.648 0.747 1.648 1.648zM12.274 3.030c0 0.927-0.747 1.648-1.648 1.648s-1.648-0.721-1.648-1.648c0-0.901 0.747-1.622 1.648-1.622s1.648 0.721 1.648 1.622z"></path></symbol><symbol id="icon-group" viewBox="0 0 16 16"><path d="M5.477 8c-0.747 0.026-1.339 0.309-1.828 0.875h-0.901c-0.386 0-0.695-0.077-0.953-0.283-0.257-0.18-0.386-0.438-0.386-0.798 0-1.622 0.283-2.42 0.85-2.42 0.026 0 0.129 0.026 0.309 0.129 0.155 0.103 0.386 0.206 0.669 0.309 0.257 0.077 0.541 0.129 0.798 0.129 0.309 0 0.618-0.051 0.927-0.154-0.026 0.18-0.026 0.335-0.026 0.463 0 0.618 0.18 1.21 0.541 1.751zM12.841 12.377c0 0.541-0.18 0.978-0.515 1.287-0.335 0.335-0.772 0.489-1.313 0.489h-6.025c-0.541 0-0.978-0.155-1.313-0.489-0.335-0.309-0.515-0.747-0.515-1.287 0-0.257 0.026-0.489 0.026-0.721 0.026-0.232 0.051-0.463 0.103-0.747 0.051-0.257 0.103-0.515 0.18-0.747s0.18-0.438 0.283-0.669c0.129-0.206 0.283-0.386 0.438-0.541s0.36-0.283 0.592-0.386c0.232-0.077 0.489-0.129 0.772-0.129 0.026 0 0.129 0.052 0.283 0.155s0.309 0.206 0.489 0.335c0.206 0.103 0.438 0.232 0.747 0.309 0.309 0.103 0.618 0.155 0.927 0.155s0.618-0.052 0.927-0.155c0.309-0.077 0.541-0.206 0.747-0.309 0.18-0.129 0.335-0.232 0.489-0.335s0.257-0.155 0.283-0.155c0.283 0 0.541 0.052 0.772 0.129 0.232 0.103 0.438 0.232 0.592 0.386s0.309 0.335 0.438 0.541c0.103 0.232 0.206 0.438 0.283 0.669s0.129 0.489 0.18 0.747c0.052 0.283 0.077 0.515 0.103 0.747 0 0.232 0.026 0.463 0.026 0.721zM5.811 3.597c0 0.489-0.18 0.901-0.515 1.262-0.36 0.335-0.772 0.515-1.262 0.515-0.463 0-0.901-0.18-1.236-0.515-0.335-0.36-0.515-0.772-0.515-1.262s0.18-0.901 0.515-1.236c0.335-0.335 0.772-0.515 1.236-0.515 0.489 0 0.901 0.18 1.262 0.515 0.335 0.335 0.515 0.747 0.515 1.236zM10.626 6.249c0 0.721-0.257 1.339-0.772 1.854s-1.133 0.772-1.854 0.772c-0.721 0-1.339-0.257-1.854-0.772s-0.772-1.133-0.772-1.854c0-0.747 0.257-1.365 0.772-1.88s1.133-0.772 1.854-0.772c0.721 0 1.339 0.257 1.854 0.772s0.772 1.133 0.772 1.88zM14.592 7.794c0 0.36-0.129 0.618-0.386 0.798-0.257 0.206-0.566 0.283-0.953 0.283h-0.901c-0.489-0.566-1.081-0.85-1.828-0.875 0.36-0.541 0.541-1.133 0.541-1.751 0-0.129 0-0.283-0.026-0.463 0.309 0.103 0.618 0.154 0.927 0.154 0.257 0 0.541-0.051 0.798-0.129 0.283-0.103 0.515-0.206 0.669-0.309 0.18-0.103 0.283-0.129 0.309-0.129 0.566 0 0.85 0.798 0.85 2.42zM13.716 3.597c0 0.489-0.18 0.901-0.515 1.262-0.335 0.335-0.772 0.515-1.236 0.515-0.489 0-0.901-0.18-1.262-0.515-0.335-0.36-0.515-0.772-0.515-1.262s0.18-0.901 0.515-1.236c0.36-0.335 0.772-0.515 1.262-0.515 0.463 0 0.901 0.18 1.236 0.515s0.515 0.747 0.515 1.236z"></path></symbol><symbol id="icon-help" viewBox="0 0 16 16"><path d="M7.912 1.086c1.904 0 3.516 0.645 4.893 1.963 1.348 1.318 2.051 2.93 2.109 4.834 0 1.904-0.645 3.545-1.992 4.893-1.318 1.377-2.93 2.080-4.834 2.139-1.904 0-3.545-0.645-4.893-1.992-1.377-1.318-2.080-2.93-2.109-4.834s0.615-3.545 1.963-4.893c1.348-1.377 2.959-2.080 4.863-2.109zM7.883 12.189c0.293 0 0.557-0.088 0.732-0.264 0.176-0.205 0.293-0.439 0.293-0.703 0-0.322-0.059-0.557-0.264-0.762-0.176-0.176-0.439-0.264-0.732-0.264 0 0-0.029 0-0.029 0-0.293 0-0.527 0.088-0.703 0.264-0.205 0.176-0.293 0.41-0.322 0.703 0 0.293 0.088 0.527 0.293 0.732 0.176 0.176 0.41 0.293 0.703 0.293 0 0 0.029 0 0.029 0zM10.373 7.268c0.264-0.322 0.381-0.703 0.381-1.172 0-0.762-0.264-1.348-0.791-1.729s-1.201-0.557-2.021-0.557c-0.645 0-1.172 0.117-1.553 0.381-0.703 0.41-1.055 1.143-1.084 2.197 0 0 0 0.059 0 0.059s1.641 0 1.641 0c0 0 0-0.059 0-0.059 0-0.264 0.088-0.527 0.234-0.82 0.176-0.234 0.439-0.352 0.82-0.352 0.41 0 0.674 0.088 0.791 0.293 0.146 0.205 0.234 0.41 0.234 0.674 0 0.176-0.088 0.381-0.234 0.586-0.088 0.117-0.205 0.234-0.322 0.293 0 0-0.088 0.059-0.088 0.059-0.059 0.059-0.117 0.117-0.234 0.176-0.088 0.059-0.205 0.146-0.293 0.234-0.117 0.059-0.205 0.146-0.322 0.234s-0.205 0.176-0.264 0.264c-0.117 0.205-0.205 0.586-0.264 1.172 0 0 0 0.117 0 0.117s1.641 0 1.641 0c0 0 0-0.059 0-0.059 0-0.117 0-0.264 0.059-0.41 0.059-0.205 0.176-0.381 0.41-0.557 0 0 0.41-0.264 0.41-0.264 0.469-0.352 0.732-0.586 0.85-0.762z"></path></symbol><symbol id="icon-history" viewBox="0 0 512 512"><path d="M504 255.531c.253 136.64-111.18 248.372-247.82 248.468-59.015.042-113.223-20.53-155.822-54.911-11.077-8.94-11.905-25.541-1.839-35.607l11.267-11.267c8.609-8.609 22.353-9.551 31.891-1.984C173.062 425.135 212.781 440 256 440c101.705 0 184-82.311 184-184 0-101.705-82.311-184-184-184-48.814 0-93.149 18.969-126.068 49.932l50.754 50.754c10.08 10.08 2.941 27.314-11.313 27.314H24c-8.837 0-16-7.163-16-16V38.627c0-14.254 17.234-21.393 27.314-11.314l49.372 49.372C129.209 34.136 189.552 8 256 8c136.81 0 247.747 110.78 248 247.531zm-180.912 78.784l9.823-12.63c8.138-10.463 6.253-25.542-4.21-33.679L288 256.349V152c0-13.255-10.745-24-24-24h-16c-13.255 0-24 10.745-24 24v135.651l65.409 50.874c10.463 8.137 25.541 6.253 33.679-4.21z"></path></symbol><symbol id="icon-home" viewBox="0 0 16 16"><path d="M12.834 8.791v4.043c0 0.146-0.059 0.264-0.176 0.352-0.088 0.117-0.234 0.176-0.381 0.176h-3.193v-3.223h-2.168v3.223h-3.193c-0.146 0-0.293-0.059-0.381-0.176-0.117-0.088-0.176-0.205-0.176-0.352v-4.043c0 0 0 0 0.029 0 0-0.029 0-0.029 0-0.029l4.805-3.984 4.805 3.984c0 0 0.029 0.029 0.029 0.029zM14.68 8.234l-0.498 0.615c-0.059 0.059-0.117 0.088-0.176 0.088h-0.029c-0.088 0-0.146-0.029-0.176-0.059l-5.801-4.834-5.801 4.834c-0.059 0.059-0.117 0.059-0.205 0.059-0.059 0-0.117-0.029-0.176-0.088l-0.498-0.615c-0.059-0.059-0.059-0.117-0.059-0.205s0.029-0.146 0.088-0.176l6.006-5.010c0.176-0.146 0.41-0.234 0.645-0.234s0.469 0.088 0.645 0.234l2.021 1.699v-1.641c0-0.059 0.029-0.146 0.088-0.176 0.059-0.059 0.117-0.088 0.205-0.088h1.582c0.088 0 0.146 0.029 0.205 0.088 0.059 0.029 0.088 0.117 0.088 0.176v3.428l1.816 1.523c0.059 0.029 0.088 0.088 0.088 0.176s0 0.146-0.059 0.205z"></path></symbol><symbol id="icon-horizontalrule" viewBox="0 0 16 16"><path d="M15.237 8.3H.764l.004.004-.268.268v-.001.838-.001l.253.252h14.496l.251-.251v-.846z"/></symbol><symbol id="icon-image" viewBox="0 0 16 16"><path d="M5.665 5.665c0 0.385-0.137 0.714-0.412 0.989s-0.604 0.412-1.016 0.412c-0.385 0-0.714-0.137-0.989-0.412s-0.412-0.604-0.412-0.989c0-0.412 0.137-0.742 0.412-1.016s0.604-0.412 0.989-0.412c0.412 0 0.742 0.137 1.016 0.412s0.412 0.604 0.412 1.016zM13.164 8.467v3.296h-10.327v-1.428l2.362-2.335 1.154 1.181 3.763-3.763zM13.85 3.303h-11.7c-0.082 0-0.137 0.027-0.165 0.082-0.055 0.055-0.082 0.11-0.082 0.165v8.899c0 0.055 0.027 0.11 0.082 0.165 0.027 0.055 0.082 0.082 0.165 0.082h11.7c0.082 0 0.137-0.027 0.165-0.082 0.055-0.055 0.082-0.11 0.082-0.165v-8.899c0-0.055-0.027-0.11-0.082-0.165-0.027-0.055-0.082-0.082-0.165-0.082zM15.031 3.551v8.899c0 0.33-0.11 0.604-0.357 0.824-0.22 0.247-0.494 0.357-0.824 0.357h-11.7c-0.33 0-0.604-0.11-0.824-0.357-0.247-0.22-0.357-0.494-0.357-0.824v-8.899c0-0.33 0.11-0.604 0.357-0.824 0.22-0.247 0.494-0.357 0.824-0.357h11.7c0.33 0 0.604 0.11 0.824 0.357 0.247 0.22 0.357 0.494 0.357 0.824z"></path></symbol><symbol id="icon-info-circle" viewBox="0 0 512 512"><path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"></path></symbol><symbol id="icon-italic" viewBox="0 0 16 16"><path d="M3.723 14.416l0.146-0.703c0-0.029 0.234-0.088 0.645-0.176 0.41-0.117 0.732-0.205 0.967-0.322 0.146-0.205 0.264-0.498 0.322-0.85l0.234-1.172 0.586-2.783c0.029-0.234 0.088-0.469 0.117-0.703 0.059-0.234 0.117-0.41 0.146-0.557s0.088-0.264 0.117-0.381c0.029-0.117 0.029-0.205 0.059-0.264s0.029-0.088 0.029-0.088l0.234-1.318 0.146-0.527 0.176-1.143 0.088-0.41v-0.322c-0.234-0.117-0.645-0.205-1.23-0.234-0.146 0-0.264-0.029-0.293-0.029l0.146-0.85 2.666 0.117c0.205 0 0.41 0 0.586 0 0.381 0 0.967-0.029 1.816-0.059 0.176-0.029 0.352-0.029 0.557-0.059 0.205 0 0.293 0 0.293 0 0 0.088-0.029 0.205-0.029 0.293-0.059 0.176-0.088 0.322-0.117 0.439-0.322 0.117-0.615 0.205-0.908 0.264-0.352 0.088-0.645 0.176-0.85 0.264-0.059 0.176-0.146 0.41-0.205 0.732-0.059 0.234-0.088 0.469-0.117 0.674-0.234 1.113-0.41 1.963-0.557 2.578l-0.498 2.607-0.322 1.318-0.352 1.963-0.117 0.381c0 0.029 0 0.117 0.029 0.205 0.352 0.088 0.674 0.146 0.996 0.176 0.205 0.029 0.381 0.059 0.557 0.088-0.029 0.176-0.029 0.322-0.059 0.498-0.059 0.176-0.088 0.293-0.088 0.352-0.088 0-0.176 0-0.205 0-0.117 0-0.234 0-0.352 0-0.029 0-0.117 0-0.234 0-0.088-0.029-0.498-0.088-1.201-0.146l-1.67-0.029c-0.205 0-0.703 0.029-1.436 0.088-0.41 0.059-0.703 0.059-0.82 0.088z"></path></symbol><symbol id="icon-link-external" viewBox="0 0 512 512"><path d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"></path></symbol><symbol id="icon-link" viewBox="0 0 16 16"><path d="M12.889 10.999c0-0.192-0.055-0.385-0.22-0.522l-1.62-1.62c-0.137-0.165-0.33-0.22-0.549-0.22s-0.385 0.082-0.549 0.247c0 0 0.055 0.055 0.137 0.137s0.137 0.137 0.165 0.165c0.027 0.027 0.082 0.082 0.137 0.165 0.055 0.055 0.082 0.11 0.082 0.192 0.027 0.055 0.027 0.137 0.027 0.22 0 0.192-0.055 0.385-0.22 0.522-0.137 0.137-0.302 0.22-0.522 0.22-0.082 0-0.137 0-0.22-0.027-0.055-0.027-0.137-0.055-0.192-0.11-0.082-0.027-0.11-0.082-0.165-0.11-0.027-0.027-0.082-0.082-0.165-0.165s-0.11-0.137-0.137-0.137c-0.165 0.137-0.247 0.33-0.247 0.549s0.055 0.412 0.22 0.549l1.593 1.62c0.165 0.137 0.33 0.22 0.549 0.22 0.192 0 0.385-0.082 0.522-0.22l1.154-1.126c0.165-0.165 0.22-0.33 0.22-0.549zM7.368 5.479c0-0.22-0.055-0.385-0.22-0.522l-1.593-1.648c-0.165-0.137-0.33-0.22-0.549-0.22-0.192 0-0.385 0.082-0.522 0.22l-1.154 1.154c-0.165 0.137-0.22 0.33-0.22 0.522 0 0.22 0.055 0.385 0.22 0.522l1.62 1.648c0.137 0.137 0.33 0.22 0.549 0.22s0.385-0.082 0.549-0.247c0-0.027-0.055-0.082-0.137-0.165s-0.137-0.137-0.165-0.165c-0.027-0.027-0.082-0.082-0.137-0.137-0.055-0.082-0.082-0.137-0.082-0.192-0.027-0.082-0.027-0.137-0.027-0.22 0-0.22 0.055-0.385 0.22-0.549 0.137-0.137 0.302-0.22 0.522-0.22 0.082 0 0.137 0.027 0.22 0.027 0.055 0.027 0.137 0.055 0.192 0.11 0.082 0.055 0.11 0.082 0.165 0.11 0.027 0.027 0.082 0.082 0.165 0.165s0.11 0.137 0.137 0.165c0.165-0.165 0.247-0.357 0.247-0.577zM14.4 10.999c0 0.632-0.22 1.181-0.659 1.593l-1.154 1.154c-0.439 0.439-0.961 0.659-1.593 0.659s-1.181-0.22-1.593-0.659l-1.62-1.648c-0.439-0.412-0.659-0.961-0.659-1.593s0.22-1.181 0.687-1.62l-0.687-0.687c-0.439 0.439-0.989 0.687-1.62 0.687s-1.181-0.22-1.62-0.659l-1.62-1.648c-0.439-0.439-0.659-0.961-0.659-1.593s0.22-1.154 0.659-1.593l1.154-1.154c0.439-0.439 0.961-0.632 1.593-0.632s1.181 0.22 1.593 0.659l1.62 1.62c0.439 0.439 0.659 0.961 0.659 1.593s-0.22 1.181-0.687 1.648l0.687 0.687c0.439-0.467 0.989-0.687 1.62-0.687s1.181 0.22 1.62 0.659l1.62 1.62c0.439 0.439 0.659 0.989 0.659 1.593z"></path></symbol><symbol id="icon-list-ol" viewBox="0 0 16 16"><path d="M4.451 12.997c0 0.362-0.121 0.652-0.386 0.869-0.241 0.217-0.555 0.314-0.917 0.314-0.507 0-0.893-0.145-1.207-0.459l0.41-0.603c0.217 0.217 0.459 0.314 0.724 0.314 0.145 0 0.241-0.048 0.338-0.097 0.121-0.072 0.169-0.169 0.169-0.29 0-0.314-0.241-0.435-0.724-0.386l-0.193-0.386c0.048-0.048 0.121-0.145 0.217-0.314 0.121-0.145 0.217-0.266 0.314-0.362 0.072-0.097 0.169-0.193 0.241-0.266v-0.024c-0.072 0-0.193 0.024-0.338 0.024s-0.266 0-0.338 0v0.362h-0.724v-1.038h2.293v0.603l-0.652 0.797c0.241 0.048 0.435 0.169 0.555 0.338 0.145 0.169 0.217 0.362 0.217 0.603zM4.451 8.676v1.086h-2.486c-0.024-0.169-0.048-0.29-0.048-0.362 0-0.241 0.072-0.459 0.169-0.652s0.241-0.338 0.386-0.459c0.145-0.121 0.314-0.241 0.459-0.338s0.29-0.193 0.386-0.29c0.121-0.097 0.169-0.217 0.169-0.314 0-0.121-0.048-0.217-0.097-0.266-0.072-0.072-0.169-0.097-0.29-0.097-0.193 0-0.386 0.145-0.555 0.41l-0.579-0.41c0.121-0.241 0.266-0.435 0.483-0.555s0.483-0.193 0.748-0.193c0.314 0 0.603 0.097 0.845 0.29 0.217 0.193 0.338 0.435 0.338 0.772 0 0.217-0.072 0.435-0.241 0.628-0.145 0.193-0.314 0.338-0.507 0.435-0.193 0.121-0.362 0.241-0.531 0.362-0.145 0.121-0.241 0.241-0.241 0.362h0.869v-0.41zM14.18 10.873v1.328c0 0.048-0.024 0.097-0.072 0.145s-0.097 0.072-0.145 0.072h-8.401c-0.048 0-0.097-0.024-0.145-0.072s-0.072-0.097-0.072-0.145v-1.328c0-0.072 0.024-0.121 0.072-0.169 0.048-0.024 0.097-0.048 0.145-0.048h8.401c0.048 0 0.097 0.024 0.145 0.048 0.048 0.048 0.072 0.097 0.072 0.169zM4.476 4.669v0.676h-2.317v-0.676h0.748c0-0.193 0-0.459 0-0.845 0-0.362 0-0.652 0-0.821v-0.097h-0.024c-0.024 0.072-0.145 0.217-0.338 0.386l-0.483-0.531 0.941-0.869h0.724v2.776zM14.18 7.348v1.304c0 0.072-0.024 0.121-0.072 0.169s-0.097 0.072-0.145 0.072h-8.401c-0.048 0-0.097-0.024-0.145-0.072s-0.072-0.097-0.072-0.169v-1.304c0-0.072 0.024-0.121 0.072-0.169s0.097-0.072 0.145-0.072h8.401c0.048 0 0.097 0.024 0.145 0.072s0.072 0.097 0.072 0.169zM14.18 3.8v1.328c0 0.072-0.024 0.121-0.072 0.169-0.048 0.024-0.097 0.048-0.145 0.048h-8.401c-0.048 0-0.097-0.024-0.145-0.048-0.048-0.048-0.072-0.097-0.072-0.169v-1.328c0-0.048 0.024-0.097 0.072-0.145s0.097-0.072 0.145-0.072h8.401c0.048 0 0.097 0.024 0.145 0.072s0.072 0.097 0.072 0.145z"></path></symbol><symbol id="icon-list-ul" viewBox="0 0 16 16"><path d="M4.241 11.759c0 0.412-0.154 0.721-0.412 1.004-0.283 0.283-0.618 0.412-1.004 0.412s-0.721-0.129-1.004-0.412c-0.283-0.283-0.412-0.592-0.412-1.004 0-0.386 0.129-0.721 0.412-1.004 0.283-0.257 0.618-0.412 1.004-0.412s0.721 0.155 1.004 0.412c0.257 0.283 0.412 0.618 0.412 1.004zM4.241 8c0 0.386-0.154 0.721-0.412 1.004-0.283 0.283-0.618 0.412-1.004 0.412s-0.721-0.129-1.004-0.412c-0.283-0.283-0.412-0.618-0.412-1.004s0.129-0.721 0.412-1.004c0.283-0.283 0.618-0.412 1.004-0.412s0.721 0.129 1.004 0.412c0.257 0.283 0.412 0.618 0.412 1.004zM14.592 11.064v1.416c0 0.052-0.026 0.103-0.077 0.155s-0.103 0.077-0.155 0.077h-8.961c-0.051 0-0.103-0.026-0.154-0.077s-0.077-0.103-0.077-0.155v-1.416c0-0.077 0.026-0.129 0.077-0.18 0.051-0.026 0.103-0.052 0.154-0.052h8.961c0.052 0 0.103 0.026 0.155 0.052 0.052 0.052 0.077 0.103 0.077 0.18zM4.241 4.241c0 0.386-0.154 0.721-0.412 1.004-0.283 0.257-0.618 0.412-1.004 0.412s-0.721-0.154-1.004-0.412c-0.283-0.283-0.412-0.618-0.412-1.004 0-0.412 0.129-0.721 0.412-1.004s0.618-0.412 1.004-0.412c0.386 0 0.721 0.129 1.004 0.412 0.257 0.283 0.412 0.592 0.412 1.004zM14.592 7.305v1.39c0 0.077-0.026 0.129-0.077 0.18s-0.103 0.077-0.155 0.077h-8.961c-0.051 0-0.103-0.026-0.154-0.077s-0.077-0.103-0.077-0.18v-1.39c0-0.077 0.026-0.129 0.077-0.18s0.103-0.077 0.154-0.077h8.961c0.052 0 0.103 0.026 0.155 0.077s0.077 0.103 0.077 0.18zM14.592 3.52v1.416c0 0.077-0.026 0.129-0.077 0.18-0.052 0.026-0.103 0.051-0.155 0.051h-8.961c-0.051 0-0.103-0.026-0.154-0.051-0.051-0.051-0.077-0.103-0.077-0.18v-1.416c0-0.051 0.026-0.103 0.077-0.155s0.103-0.077 0.154-0.077h8.961c0.052 0 0.103 0.026 0.155 0.077s0.077 0.103 0.077 0.155z"></path></symbol><symbol id="icon-lock-open" viewBox="0 0 16 16"><path d="M12.593 8.548c-0.159-0.061-0.558-0.2-0.564-0.205-0.467-0.117-1.4-0.377-1.4-0.377-0.851-0.183-1.725-0.276-2.596-0.276-0.846 0-1.694 0.088-2.524 0.261l-1.297-1.297c-1.012-1.012-1.012-2.659 0-3.672s2.659-1.012 3.672-0l1.349 1.349c0.455-0.11 0.963-0.334 1.286-0.694-0.026 0.003-0.061-0.005-0.075-0.019l-1.598-1.598c-1.543-1.543-4.053-1.543-5.596 0s-1.543 4.053 0 5.596l0.746 0.746c-0.203 0.068-0.37 0.127-0.523 0.187-0.19 0.073-0.333 0.283-0.333 0.489v5.192c0 0.204 0.142 0.415 0.331 0.489 1.458 0.578 2.993 0.871 4.562 0.871s3.104-0.293 4.562-0.871c0.188-0.075 0.331-0.285 0.331-0.489v-5.192c0-0.206-0.143-0.416-0.333-0.489zM8.571 11.531v1.782c0 0.298-0.241 0.539-0.539 0.539s-0.539-0.241-0.539-0.539v-1.782c-0.316-0.187-0.539-0.518-0.539-0.912 0-0.596 0.483-1.078 1.078-1.078s1.078 0.482 1.078 1.078c0 0.395-0.223 0.725-0.539 0.912z"></path></symbol><symbol id="icon-lock" viewBox="0 0 16 16"><path d="M8.047 0.762c-2.385 0-4.325 1.94-4.325 4.325v2.47c0 0.028-0.035 0.075-0.062 0.084-0.234 0.078-0.424 0.146-0.598 0.213-0.207 0.080-0.364 0.31-0.364 0.534v5.676c0 0.223 0.155 0.453 0.361 0.535 1.594 0.632 3.272 0.952 4.987 0.952s3.393-0.32 4.987-0.952c0.206-0.082 0.361-0.312 0.361-0.535v-5.676c0-0.225-0.156-0.455-0.364-0.535-0.174-0.067-0.364-0.135-0.598-0.213-0.027-0.009-0.062-0.056-0.062-0.084v-2.47c-0-2.385-1.94-4.325-4.325-4.325zM6.869 10.117c0-0.651 0.527-1.178 1.178-1.178s1.178 0.527 1.178 1.178c0 0.431-0.243 0.793-0.589 0.997v1.948c0 0.326-0.263 0.589-0.589 0.589s-0.589-0.263-0.589-0.589v-1.948c-0.346-0.205-0.589-0.566-0.589-0.997zM10.885 5.087v2.131c-0.931-0.2-1.885-0.302-2.837-0.302s-1.906 0.101-2.838 0.302v-2.131c0-1.565 1.273-2.838 2.838-2.838s2.838 1.273 2.838 2.838z"></path></symbol><symbol id="icon-login" viewBox="0 0 512 512"><path d="M416 448h-84c-6.6 0-12-5.4-12-12v-40c0-6.6 5.4-12 12-12h84c17.7 0 32-14.3 32-32V160c0-17.7-14.3-32-32-32h-84c-6.6 0-12-5.4-12-12V76c0-6.6 5.4-12 12-12h84c53 0 96 43 96 96v192c0 53-43 96-96 96zm-47-201L201 79c-15-15-41-4.5-41 17v96H24c-13.3 0-24 10.7-24 24v96c0 13.3 10.7 24 24 24h136v96c0 21.5 26 32 41 17l168-168c9.3-9.4 9.3-24.6 0-34z"></path></symbol><symbol id="icon-logout" viewBox="0 0 16 16"><path d="M14.943 0h-9.41c-0.564 0-1.022 0.458-1.022 1.022s0.458 1.022 1.022 1.022h8.388v11.841h-8.388c-0.564 0-1.022 0.458-1.022 1.022s0.458 1.022 1.022 1.022h9.41c0.564 0 1.022-0.458 1.022-1.022v-13.885c0.035-0.564-0.458-1.022-1.022-1.022z"></path><path d="M5.040 10.573l-1.551-1.551h7.366c0.564 0 1.022-0.458 1.022-1.022s-0.458-1.022-1.022-1.022h-7.401l1.551-1.551c0.388-0.388 0.388-1.022 0-1.445-0.176-0.176-0.458-0.282-0.705-0.282-0.282 0-0.529 0.106-0.705 0.282l-3.313 3.278c-0.106 0.141-0.211 0.282-0.247 0.493v0.070c-0.035 0.035-0.035 0.106-0.035 0.176s0 0.141 0 0.176v0.070c0.035 0.176 0.141 0.352 0.247 0.458l3.313 3.278c0.176 0.176 0.458 0.282 0.705 0.282 0.282 0 0.529-0.106 0.705-0.282 0.458-0.388 0.458-1.022 0.070-1.41z"></path></symbol><symbol id="icon-mail" viewBox="0 0 16 16"><path d="M15.031 6.044v6.235c0 0.33-0.11 0.632-0.357 0.879s-0.549 0.357-0.906 0.357h-11.536c-0.357 0-0.659-0.11-0.906-0.357s-0.357-0.549-0.357-0.879v-6.235c0.22 0.247 0.494 0.467 0.797 0.687 1.895 1.291 3.186 2.17 3.9 2.692 0.302 0.22 0.549 0.412 0.714 0.522 0.192 0.11 0.439 0.247 0.742 0.385 0.33 0.11 0.604 0.192 0.879 0.192v0c0.275 0 0.549-0.082 0.879-0.192 0.302-0.137 0.549-0.275 0.742-0.385 0.165-0.11 0.412-0.302 0.714-0.522 0.906-0.632 2.197-1.538 3.9-2.692 0.302-0.22 0.577-0.439 0.797-0.687zM15.031 3.737c0 0.412-0.137 0.797-0.385 1.181s-0.577 0.687-0.961 0.961c-1.95 1.373-3.186 2.225-3.68 2.554-0.027 0.027-0.165 0.11-0.33 0.247-0.165 0.11-0.302 0.22-0.412 0.302-0.11 0.055-0.247 0.137-0.412 0.247-0.165 0.082-0.302 0.165-0.439 0.22-0.165 0.027-0.275 0.055-0.412 0.055v0c-0.137 0-0.247-0.027-0.412-0.055-0.137-0.055-0.275-0.137-0.439-0.22-0.165-0.11-0.302-0.192-0.412-0.247-0.11-0.082-0.247-0.192-0.412-0.302-0.165-0.137-0.302-0.22-0.33-0.247-0.494-0.33-1.181-0.824-2.060-1.428-0.906-0.632-1.428-0.989-1.62-1.126-0.33-0.22-0.632-0.522-0.906-0.906-0.302-0.385-0.439-0.742-0.439-1.071 0-0.412 0.11-0.742 0.33-1.016s0.522-0.412 0.934-0.412h11.536c0.357 0 0.632 0.137 0.879 0.385s0.385 0.522 0.385 0.879z"></path></symbol><symbol id="icon-media" viewBox="0 0 16 16"><path d="M11.719 8c0 0.219-0.094 0.375-0.281 0.5l-4.875 2.844c-0.094 0.063-0.188 0.094-0.281 0.094s-0.188-0.031-0.281-0.094-0.281-0.281-0.281-0.5v-5.688c0-0.219 0.094-0.406 0.281-0.5 0.188-0.125 0.375-0.125 0.563 0l4.875 2.844c0.188 0.125 0.281 0.281 0.281 0.5zM12.844 8c0-0.875-0.219-1.688-0.625-2.438-0.438-0.75-1.031-1.344-1.781-1.781-0.75-0.406-1.563-0.625-2.438-0.625s-1.688 0.219-2.438 0.625c-0.75 0.438-1.344 1.031-1.781 1.781-0.406 0.75-0.625 1.563-0.625 2.438s0.219 1.688 0.625 2.438c0.438 0.75 1.031 1.344 1.781 1.781 0.75 0.406 1.563 0.625 2.438 0.625s1.688-0.219 2.438-0.625c0.75-0.438 1.344-1.031 1.781-1.781 0.406-0.75 0.625-1.563 0.625-2.438zM14.844 8c0 1.25-0.281 2.406-0.906 3.438-0.625 1.063-1.438 1.875-2.5 2.5-1.031 0.625-2.188 0.906-3.438 0.906s-2.406-0.281-3.438-0.906c-1.063-0.625-1.875-1.438-2.5-2.5-0.625-1.031-0.906-2.188-0.906-3.438s0.281-2.406 0.906-3.438c0.625-1.063 1.438-1.875 2.5-2.5 1.031-0.625 2.188-0.906 3.438-0.906s2.406 0.281 3.438 0.906c1.063 0.625 1.875 1.438 2.5 2.5 0.625 1.031 0.906 2.188 0.906 3.438z"></path></symbol><symbol id="icon-no-view" viewBox="0 0 16 16"><path d="M0.969 8.002q0-0.298 0.157-0.541 1.201-1.844 2.982-2.911t3.892-1.067q0.698 0 1.413 0.133l0.424-0.761q0.078-0.126 0.22-0.126 0.039 0 0.141 0.047t0.243 0.122 0.259 0.145 0.247 0.145 0.153 0.090q0.126 0.078 0.126 0.212 0 0.055-0.008 0.071-0.824 1.475-2.472 4.442t-2.48 4.449l-0.385 0.698q-0.078 0.126-0.22 0.126-0.094 0-1.052-0.549-0.126-0.078-0.126-0.22 0-0.094 0.345-0.683-1.122-0.51-2.068-1.358t-1.636-1.923q-0.157-0.243-0.157-0.541zM1.973 8.002q1.311 2.025 3.351 2.943l0.612-1.106q-0.683-0.494-1.067-1.248t-0.385-1.593q0-0.95 0.479-1.766-1.797 0.918-2.99 2.77zM5.614 6.998q0 0.157 0.11 0.267t0.267 0.11 0.267-0.11 0.11-0.267q0-0.675 0.479-1.154t1.154-0.479q0.157 0 0.267-0.11t0.11-0.267-0.11-0.267-0.267-0.11q-0.981 0-1.683 0.702t-0.702 1.683zM8 12.522l0.581-1.036q1.664-0.141 3.080-1.075t2.366-2.409q-0.902-1.405-2.213-2.307l0.494-0.879q0.745 0.502 1.432 1.201t1.134 1.444q0.157 0.267 0.157 0.541t-0.157 0.541q-0.306 0.502-0.855 1.138-1.177 1.35-2.727 2.095t-3.292 0.745zM9.256 10.278l2.197-3.939q0.063 0.353 0.063 0.659 0 1.091-0.62 1.989t-1.64 1.291z"></path></symbol><symbol id="icon-openquote" viewBox="0 0 16 16"><path d="M3.516 7c1.933 0 3.5 1.567 3.5 3.5s-1.567 3.5-3.5 3.5-3.5-1.567-3.5-3.5l-0.016-0.5c0-3.866 3.134-7 7-7v2c-1.336 0-2.591 0.52-3.536 1.464-0.182 0.182-0.348 0.375-0.497 0.578 0.179-0.028 0.362-0.043 0.548-0.043zM12.516 7c1.933 0 3.5 1.567 3.5 3.5s-1.567 3.5-3.5 3.5-3.5-1.567-3.5-3.5l-0.016-0.5c0-3.866 3.134-7 7-7v2c-1.336 0-2.591 0.52-3.536 1.464-0.182 0.182-0.348 0.375-0.497 0.578 0.179-0.028 0.362-0.043 0.549-0.043z"/></symbol><symbol id="icon-order-down" viewBox="0 0 16 16"><path d="M10.994 11.159c0.027 0.11 0.027 0.192-0.055 0.275l-2.747 3.021c-0.027 0.055-0.11 0.082-0.165 0.082-0.082 0-0.137-0.027-0.192-0.082l-2.774-3.021c-0.082-0.082-0.082-0.165-0.055-0.275 0.055-0.082 0.137-0.137 0.22-0.137h1.758v-9.805c0-0.082 0.027-0.137 0.082-0.165 0.055-0.055 0.11-0.082 0.192-0.082h1.483c0.082 0 0.137 0.027 0.192 0.082 0.055 0.027 0.082 0.082 0.082 0.165v9.805h1.758c0.11 0 0.165 0.055 0.22 0.137z"></path></symbol><symbol id="icon-order-up" viewBox="0 0 16 16"><path d="M10.994 4.841c-0.055 0.082-0.137 0.137-0.22 0.137h-1.758v9.805c0 0.082-0.027 0.137-0.082 0.165-0.055 0.055-0.11 0.082-0.192 0.082h-1.483c-0.082 0-0.137-0.027-0.192-0.082-0.055-0.027-0.082-0.082-0.082-0.165v-9.805h-1.758c-0.11 0-0.165-0.055-0.22-0.137-0.027-0.11-0.027-0.192 0.055-0.275l2.747-3.021c0.027-0.055 0.11-0.082 0.165-0.082 0.082 0 0.137 0.027 0.192 0.082l2.774 3.021c0.082 0.082 0.082 0.165 0.055 0.275z"></path></symbol><symbol id="icon-order" viewBox="0 0 16 16"><path d="M11.983 10.873c0-0.29-0.097-0.555-0.29-0.797-0.217-0.241-0.459-0.362-0.724-0.362-0.241 0-0.435 0.072-0.579 0.241-0.121 0.169-0.193 0.386-0.193 0.652s0.072 0.483 0.241 0.652c0.169 0.193 0.41 0.266 0.724 0.266 0.217 0 0.41-0.048 0.579-0.169 0.169-0.145 0.241-0.29 0.241-0.483zM7.783 11.742c0 0.072-0.024 0.121-0.072 0.169l-2.197 2.197c-0.048 0.048-0.097 0.072-0.169 0.072-0.048 0-0.097-0.024-0.145-0.072l-2.221-2.197c-0.072-0.072-0.072-0.145-0.048-0.241 0.048-0.097 0.121-0.145 0.217-0.145h1.328v-9.487c0-0.072 0.024-0.121 0.048-0.145 0.048-0.048 0.097-0.072 0.169-0.072h1.328c0.048 0 0.121 0.024 0.145 0.072 0.048 0.024 0.072 0.072 0.072 0.145v9.487h1.328c0.048 0 0.121 0.024 0.145 0.072 0.048 0.048 0.072 0.097 0.072 0.145zM12.949 11.283c0 0.29-0.024 0.555-0.097 0.821-0.048 0.29-0.145 0.555-0.266 0.797-0.145 0.241-0.29 0.459-0.483 0.652-0.169 0.193-0.41 0.338-0.676 0.459s-0.555 0.169-0.869 0.169c-0.29 0-0.531-0.048-0.748-0.121-0.121-0.024-0.217-0.072-0.29-0.097l0.266-0.772c0.072 0.024 0.145 0.048 0.217 0.072 0.169 0.048 0.338 0.097 0.507 0.097 0.386 0 0.7-0.145 0.941-0.41 0.217-0.266 0.386-0.603 0.459-1.014h-0.024c-0.097 0.121-0.241 0.193-0.41 0.266-0.193 0.072-0.386 0.097-0.603 0.097-0.483 0-0.869-0.169-1.183-0.483-0.314-0.338-0.459-0.724-0.459-1.207s0.169-0.893 0.483-1.231c0.338-0.314 0.748-0.483 1.255-0.483 0.579 0 1.038 0.217 1.424 0.652 0.362 0.435 0.555 1.014 0.555 1.738zM12.756 6.334v0.772h-3.235v-0.772h1.135v-2.993c0-0.024 0-0.072 0-0.121 0.024-0.048 0.024-0.097 0.024-0.121v-0.097h-0.024l-0.048 0.072c-0.048 0.048-0.097 0.121-0.169 0.217l-0.435 0.386-0.579-0.579 1.328-1.279h0.845v4.514z"></path></symbol><symbol id="icon-password" viewBox="0 0 16 16"><path d="M8.006 4.979c0-0.412-0.165-0.769-0.439-1.071-0.302-0.275-0.659-0.439-1.071-0.439s-0.769 0.165-1.071 0.439c-0.302 0.302-0.439 0.659-0.439 1.071 0 0.22 0.055 0.439 0.137 0.659-0.192-0.11-0.412-0.137-0.632-0.137-0.439 0-0.769 0.137-1.071 0.439-0.302 0.275-0.439 0.632-0.439 1.044 0 0.439 0.137 0.797 0.439 1.071 0.302 0.302 0.632 0.439 1.071 0.439 0.412 0 0.769-0.137 1.071-0.439 0.275-0.275 0.439-0.632 0.439-1.071 0-0.22-0.055-0.412-0.165-0.632 0.22 0.082 0.439 0.137 0.659 0.137 0.412 0 0.769-0.137 1.071-0.439 0.275-0.302 0.439-0.632 0.439-1.071zM14.68 10.499c0 0.11-0.137 0.275-0.385 0.522-0.247 0.275-0.439 0.385-0.522 0.385-0.055 0-0.11-0.027-0.22-0.11s-0.192-0.165-0.302-0.275c-0.082-0.082-0.165-0.192-0.302-0.302-0.11-0.137-0.165-0.192-0.192-0.22l-0.742 0.769 1.73 1.73c0.137 0.137 0.22 0.33 0.22 0.522 0 0.22-0.11 0.439-0.302 0.632-0.22 0.22-0.412 0.302-0.632 0.302s-0.412-0.055-0.549-0.22l-5.273-5.246c-0.906 0.687-1.868 1.016-2.856 1.016-0.851 0-1.538-0.275-2.087-0.797-0.522-0.549-0.797-1.236-0.797-2.087 0-0.824 0.247-1.648 0.742-2.444 0.494-0.824 1.154-1.456 1.95-1.95s1.62-0.742 2.444-0.742c0.851 0 1.566 0.247 2.087 0.797 0.549 0.522 0.824 1.236 0.824 2.087 0 0.989-0.357 1.923-1.044 2.856l2.802 2.774 0.742-0.742c-0.027-0.027-0.082-0.082-0.192-0.192-0.137-0.11-0.247-0.22-0.33-0.302s-0.165-0.192-0.247-0.275c-0.082-0.11-0.137-0.192-0.137-0.247 0-0.082 0.137-0.247 0.385-0.494 0.247-0.275 0.439-0.385 0.522-0.385 0.055 0 0.137 0.027 0.192 0.055 0.027 0.055 0.137 0.165 0.357 0.357 0.192 0.192 0.412 0.412 0.632 0.632s0.439 0.439 0.687 0.659c0.22 0.247 0.412 0.439 0.577 0.632 0.137 0.165 0.22 0.275 0.22 0.302z"></path></symbol><symbol id="icon-pick" viewBox="0 0 16 16"><path d="M6.352 6.352h-4.944l3.837 2.807-1.39 4.609 3.94-2.755 4.326 2.755-1.777-4.609 4.249-2.807h-4.944l-1.854-4.738z"></path></symbol><symbol id="icon-pilcrow" viewBox="0 0 16 16"><path d="M6 0h8v2h-2v14h-2v-14h-2v14h-2v-8c-2.209 0-4-1.791-4-4s1.791-4 4-4z"></path></symbol><symbol id="icon-placeholder" viewBox="0 0 16 16"><path d="M2 5v2h-2v-2h2zM0 2c0-1.104 0.896-2 2-2h1v2c-0.553 0-1 0.447-1 1h-2v-1zM5 2v-2h2v2h-2zM2 13c0 0.553 0.447 1 1 1v2h-1c-1.104 0-2-0.896-2-2v-1h2zM2 9v2h-2v-2h2zM5 16v-2h2v2h-2zM14 7v-2h2v2h-2zM9 16v-2h2v2h-2zM14 0c1.104 0 2 0.896 2 2v1h-2c0-0.553-0.447-1-1-1v-2h1zM14 11v-2h2v2h-2zM13 14c0.553 0 1-0.447 1-1h2v1c0 1.104-0.896 2-2 2h-1v-2zM9 2v-2h2v2h-2z"></path></symbol><symbol id="icon-plus-inverse" viewBox="0 0 16 16"><path d="M12 8.563v-1.125c0-0.156-0.063-0.313-0.156-0.406-0.125-0.125-0.25-0.188-0.406-0.188h-2.281v-2.281c0-0.156-0.063-0.281-0.188-0.406-0.094-0.094-0.25-0.156-0.406-0.156h-1.125c-0.156 0-0.313 0.063-0.406 0.156-0.125 0.125-0.188 0.25-0.188 0.406v2.281h-2.281c-0.156 0-0.281 0.063-0.406 0.188-0.094 0.094-0.156 0.25-0.156 0.406v1.125c0 0.156 0.063 0.313 0.156 0.406 0.125 0.125 0.25 0.188 0.406 0.188h2.281v2.281c0 0.156 0.063 0.281 0.188 0.406 0.094 0.094 0.25 0.156 0.406 0.156h1.125c0.156 0 0.313-0.063 0.406-0.156 0.125-0.125 0.188-0.25 0.188-0.406v-2.281h2.281c0.156 0 0.281-0.063 0.406-0.188 0.094-0.094 0.156-0.25 0.156-0.406zM14.844 8c0 1.25-0.281 2.406-0.906 3.438-0.625 1.063-1.438 1.875-2.5 2.5-1.031 0.625-2.188 0.906-3.438 0.906s-2.406-0.281-3.438-0.906c-1.063-0.625-1.875-1.438-2.5-2.5-0.625-1.031-0.906-2.188-0.906-3.438s0.281-2.406 0.906-3.438c0.625-1.063 1.438-1.875 2.5-2.5 1.031-0.625 2.188-0.906 3.438-0.906s2.406 0.281 3.438 0.906c1.063 0.625 1.875 1.438 2.5 2.5 0.625 1.031 0.906 2.188 0.906 3.438z"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="M13.889 6.652v1.611c0 0.234-0.088 0.41-0.234 0.586-0.146 0.146-0.352 0.234-0.557 0.234h-3.486v3.457c0 0.234-0.088 0.439-0.234 0.586s-0.352 0.234-0.586 0.234h-1.582c-0.234 0-0.439-0.088-0.586-0.234s-0.234-0.352-0.234-0.586v-3.457h-3.486c-0.205 0-0.41-0.088-0.557-0.234-0.146-0.176-0.234-0.352-0.234-0.586v-1.611c0-0.205 0.088-0.41 0.234-0.557s0.352-0.234 0.557-0.234h3.486v-3.486c0-0.234 0.088-0.41 0.234-0.557 0.146-0.176 0.352-0.234 0.586-0.234h1.582c0.234 0 0.439 0.059 0.586 0.234 0.146 0.146 0.234 0.322 0.234 0.557v3.486h3.486c0.205 0 0.41 0.088 0.557 0.234s0.234 0.352 0.234 0.557z"></path></symbol><symbol id="icon-radio-empty" viewBox="0 0 16 16"><path d="M8 3.156c-0.875 0-1.688 0.219-2.438 0.625-0.75 0.438-1.344 1.031-1.781 1.781-0.406 0.75-0.625 1.563-0.625 2.438s0.219 1.688 0.625 2.438c0.438 0.75 1.031 1.344 1.781 1.781 0.75 0.406 1.563 0.625 2.438 0.625s1.688-0.219 2.438-0.625c0.75-0.438 1.344-1.031 1.781-1.781 0.406-0.75 0.625-1.563 0.625-2.438s-0.219-1.688-0.625-2.438c-0.438-0.75-1.031-1.344-1.781-1.781-0.75-0.406-1.563-0.625-2.438-0.625zM14.844 8c0 1.25-0.281 2.406-0.906 3.438-0.625 1.063-1.438 1.875-2.5 2.5-1.031 0.625-2.188 0.906-3.438 0.906s-2.406-0.281-3.438-0.906c-1.063-0.625-1.875-1.438-2.5-2.5-0.625-1.031-0.906-2.188-0.906-3.438s0.281-2.406 0.906-3.438c0.625-1.063 1.438-1.875 2.5-2.5 1.031-0.625 2.188-0.906 3.438-0.906s2.406 0.281 3.438 0.906c1.063 0.625 1.875 1.438 2.5 2.5 0.625 1.031 0.906 2.188 0.906 3.438z"></path></symbol><symbol id="icon-radio-full" viewBox="0 0 16 16"><path d="M14.844 8c0 1.25-0.281 2.406-0.906 3.438-0.625 1.063-1.438 1.875-2.5 2.5-1.031 0.625-2.188 0.906-3.438 0.906s-2.406-0.281-3.438-0.906c-1.063-0.625-1.875-1.438-2.5-2.5-0.625-1.031-0.906-2.188-0.906-3.438s0.281-2.406 0.906-3.438c0.625-1.063 1.438-1.875 2.5-2.5 1.031-0.625 2.188-0.906 3.438-0.906s2.406 0.281 3.438 0.906c1.063 0.625 1.875 1.438 2.5 2.5 0.625 1.031 0.906 2.188 0.906 3.438z"></path></symbol><symbol id="icon-redirect" viewBox="0 0 16 16"><path d="M10.227 3.020v2.754h1.436v2.871h-5.625v-1.611l-4.453 2.988 4.453 2.959v-1.582h8.379v-8.379z"></path></symbol><symbol id="icon-repeat" viewBox="0 0 16 16"><path d="M14.416 2.639v3.75c0 0.146-0.029 0.264-0.146 0.381s-0.234 0.146-0.381 0.146h-3.75c-0.234 0-0.381-0.088-0.498-0.322-0.088-0.205-0.059-0.41 0.117-0.586l1.172-1.143c-0.82-0.762-1.816-1.143-2.93-1.143-0.586 0-1.143 0.117-1.67 0.322-0.527 0.234-0.967 0.527-1.348 0.938-0.41 0.381-0.703 0.82-0.938 1.348-0.205 0.527-0.322 1.084-0.322 1.67s0.117 1.143 0.322 1.67c0.234 0.527 0.527 0.967 0.938 1.348 0.381 0.41 0.82 0.703 1.348 0.938 0.527 0.205 1.084 0.322 1.67 0.322 0.674 0 1.289-0.146 1.875-0.439 0.586-0.264 1.113-0.674 1.494-1.23 0.059-0.029 0.117-0.088 0.205-0.088s0.146 0.029 0.205 0.088l1.143 1.143c0.059 0.059 0.088 0.088 0.088 0.176 0 0.059-0.029 0.117-0.059 0.176-0.615 0.732-1.348 1.318-2.227 1.729-0.85 0.41-1.758 0.586-2.725 0.586-0.879 0-1.699-0.146-2.49-0.498s-1.494-0.791-2.051-1.377c-0.586-0.557-1.025-1.26-1.377-2.051s-0.498-1.611-0.498-2.49c0-0.879 0.146-1.699 0.498-2.49s0.791-1.494 1.377-2.051c0.557-0.586 1.26-1.025 2.051-1.377s1.611-0.498 2.49-0.498c0.82 0 1.611 0.146 2.373 0.439 0.762 0.322 1.465 0.762 2.051 1.318l1.084-1.084c0.176-0.176 0.352-0.205 0.586-0.117 0.234 0.117 0.322 0.264 0.322 0.498z"></path></symbol><symbol id="icon-reset" viewBox="0 0 512 512"><path d="M137.2 110.3l21.9-21.9c9.3-9.3 24.5-9.4 33.9-.1L344.9 239c9.5 9.4 9.5 24.7 0 34.1L193 423.7c-9.4 9.3-24.5 9.3-33.9-.1l-21.9-21.9c-9.7-9.7-9.3-25.4.8-34.7l77.6-71.1H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h191.5l-77.6-71.1c-10-9.1-10.4-24.9-.7-34.5zM512 352V160c0-53-43-96-96-96h-84c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h84c17.7 0 32 14.3 32 32v192c0 17.7-14.3 32-32 32h-84c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h84c53 0 96-43 96-96z"></path></symbol><symbol id="icon-resubmit" viewBox="0 0 20 16"><g fill-rule="nonzero"><path d="M0 3.77V3.23c0-.446.399-.808.89-.808h13.36V.808c0-.719.961-1.078 1.52-.57l2.97 2.691a.755.755 0 010 1.142l-2.97 2.692c-.557.505-1.52.15-1.52-.571V4.577H.89C.4 4.577 0 4.215 0 3.769z"/><path d="M12.39 8.75H3.25V7.584c0-.518-.657-.78-1.04-.412L.178 9.116a.566.566 0 000 .824l2.032 1.944c.381.365 1.04.109 1.04-.412v-1.166h9.14c.337 0 .61-.262.61-.584v-.388a.597.597 0 00-.61-.584z" opacity=".5"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M10.666 6.973c0-1.055-0.352-1.934-1.084-2.666s-1.611-1.084-2.666-1.084c-1.025 0-1.904 0.352-2.637 1.084s-1.113 1.611-1.113 2.666c0 1.025 0.381 1.904 1.113 2.637s1.611 1.113 2.637 1.113c1.055 0 1.934-0.381 2.666-1.113s1.084-1.611 1.084-2.637zM14.973 13.916c0 0.293-0.117 0.557-0.322 0.762s-0.469 0.322-0.762 0.322c-0.293 0-0.557-0.117-0.762-0.322l-2.871-2.871c-0.996 0.703-2.109 1.055-3.34 1.055-0.791 0-1.553-0.146-2.285-0.469s-1.348-0.732-1.875-1.26c-0.527-0.527-0.938-1.143-1.26-1.875-0.293-0.732-0.469-1.494-0.469-2.285 0-0.82 0.176-1.582 0.469-2.285 0.322-0.732 0.732-1.377 1.26-1.904s1.143-0.938 1.875-1.26c0.732-0.293 1.494-0.439 2.285-0.439 0.82 0 1.582 0.146 2.314 0.439 0.703 0.322 1.348 0.732 1.875 1.26s0.938 1.172 1.26 1.904c0.293 0.703 0.469 1.465 0.469 2.285 0 1.23-0.352 2.344-1.055 3.34l2.871 2.871c0.205 0.205 0.322 0.439 0.322 0.732z"></path></symbol><symbol id="icon-site" viewBox="0 0 16 16"><path d="M1 8q0-1.866 0.92-3.442t2.496-2.496 3.442-0.92 3.442 0.92 2.496 2.496 0.92 3.442-0.92 3.442-2.496 2.496-3.442 0.92-3.442-0.92-2.496-2.496-0.92-3.442zM3.018 4.973q0.063 0.063 0.107 0.071 0.036 0.009 0.045 0.080t0.022 0.098 0.103-0.027q0.080 0.071 0.027 0.17 0.009-0.009 0.393 0.241 0.17 0.152 0.188 0.188 0.027 0.098-0.089 0.161-0.009-0.018-0.080-0.080t-0.080-0.036q-0.027 0.045 0.004 0.165t0.094 0.112q-0.063 0-0.085 0.143t-0.022 0.317-0.009 0.21l0.018 0.009q-0.027 0.107 0.049 0.308t0.192 0.174q-0.116 0.027 0.179 0.384 0.054 0.071 0.071 0.080 0.027 0.018 0.107 0.067t0.134 0.089 0.089 0.094q0.036 0.045 0.089 0.201t0.125 0.21q-0.018 0.054 0.085 0.179t0.094 0.205q-0.009 0-0.022 0.009t-0.022 0.009q0.027 0.063 0.138 0.125t0.138 0.116q0.009 0.027 0.018 0.089t0.027 0.098 0.071 0.018q0.018-0.179-0.214-0.554-0.134-0.223-0.152-0.259-0.027-0.045-0.049-0.138t-0.040-0.129q0.018 0 0.054 0.013t0.076 0.031 0.067 0.036 0.018 0.027q-0.027 0.063 0.018 0.156t0.107 0.165 0.152 0.17 0.107 0.116q0.054 0.054 0.125 0.174t0 0.121q0.080 0 0.179 0.089t0.152 0.179q0.045 0.071 0.071 0.232t0.045 0.214q0.018 0.063 0.076 0.121t0.112 0.085l0.143 0.071t0.116 0.063q0.045 0.018 0.165 0.094t0.192 0.103q0.089 0.036 0.143 0.036t0.129-0.022 0.121-0.031q0.134-0.018 0.259 0.134t0.188 0.188q0.321 0.17 0.491 0.098-0.018 0.009 0.004 0.067t0.071 0.138 0.080 0.129 0.049 0.076q0.045 0.054 0.161 0.134t0.161 0.134q0.054-0.036 0.063-0.080-0.027 0.071 0.063 0.179t0.161 0.089q0.125-0.027 0.125-0.286-0.277 0.134-0.438-0.161 0-0.009-0.022-0.049t-0.036-0.076-0.022-0.076 0-0.067 0.045-0.027q0.080 0 0.089-0.031t-0.018-0.112-0.036-0.116q-0.009-0.071-0.098-0.179t-0.107-0.134q-0.045 0.080-0.143 0.071t-0.143-0.080q0 0.009-0.013 0.049t-0.013 0.058q-0.116 0-0.134-0.009 0.009-0.027 0.022-0.156t0.031-0.201q0.009-0.036 0.049-0.107t0.067-0.129 0.036-0.112-0.040-0.085-0.156-0.022q-0.17 0.009-0.232 0.179-0.009 0.027-0.027 0.094t-0.045 0.103-0.080 0.063q-0.063 0.027-0.214 0.018t-0.214-0.045q-0.116-0.071-0.201-0.259t-0.085-0.33q0-0.089 0.022-0.237t0.027-0.223-0.049-0.219q0.027-0.018 0.080-0.085t0.089-0.094q0.018-0.009 0.040-0.013t0.040 0 0.036-0.013 0.027-0.054q-0.009-0.009-0.036-0.027-0.027-0.027-0.036-0.027 0.063 0.027 0.254-0.013t0.246 0.013q0.134 0.098 0.196-0.018 0-0.009-0.022-0.085t-0.004-0.121q0.045 0.241 0.259 0.080 0.027 0.027 0.138 0.045t0.156 0.045q0.027 0.018 0.063 0.049t0.049 0.040 0.045-0.004 0.076-0.058q0.089 0.125 0.107 0.214 0.098 0.357 0.17 0.393 0.063 0.027 0.098 0.018t0.040-0.085 0-0.125-0.013-0.112l-0.009-0.071v-0.161l-0.009-0.071q-0.134-0.027-0.165-0.107t0.013-0.165 0.134-0.165q0.009-0.009 0.071-0.031t0.138-0.058 0.112-0.071q0.188-0.17 0.134-0.313 0.063 0 0.098-0.080-0.009 0-0.045-0.027t-0.067-0.045-0.040-0.018q0.080-0.045 0.018-0.143 0.045-0.027 0.067-0.098t0.067-0.089q0.080 0.107 0.188 0.018 0.063-0.071 0.009-0.143 0.045-0.063 0.183-0.094t0.165-0.085q0.063 0.018 0.071-0.018t0.009-0.107 0.027-0.107q0.036-0.045 0.134-0.080t0.116-0.045l0.152-0.098q0.027-0.036 0-0.036 0.161 0.018 0.277-0.098 0.089-0.098-0.054-0.179 0.027-0.054-0.027-0.085t-0.134-0.049q0.027-0.009 0.103-0.004t0.094-0.013q0.134-0.089-0.063-0.143-0.152-0.045-0.384 0.107-0.018 0.009-0.085 0.085t-0.121 0.085q0.018 0 0.040-0.045t0.045-0.098 0.031-0.063q0.054-0.063 0.196-0.134 0.125-0.054 0.464-0.107 0.304-0.071 0.455 0.098-0.018-0.018 0.085-0.116t0.129-0.107q0.027-0.018 0.134-0.040t0.134-0.067l0.018-0.196q-0.107 0.009-0.156-0.063t-0.058-0.188q0 0.018-0.054 0.071 0-0.063-0.040-0.071t-0.103 0.009-0.080 0.009q-0.089-0.027-0.134-0.067t-0.071-0.147-0.036-0.134q-0.018-0.045-0.085-0.094t-0.085-0.094q-0.009-0.018-0.022-0.049t-0.027-0.058-0.036-0.049-0.049-0.022-0.063 0.045-0.067 0.089-0.040 0.045q-0.027-0.018-0.054-0.013t-0.040 0.009-0.040 0.027-0.045 0.031q-0.027 0.018-0.076 0.027t-0.076 0.018q0.134-0.045-0.009-0.098-0.089-0.036-0.143-0.027 0.080-0.036 0.067-0.107t-0.076-0.125h0.045q-0.009-0.036-0.076-0.076t-0.156-0.076-0.116-0.054q-0.071-0.045-0.304-0.085t-0.295-0.004q-0.045 0.054-0.040 0.094t0.036 0.125 0.031 0.112q0.009 0.054-0.049 0.116t-0.058 0.107q0 0.063 0.125 0.138t0.089 0.192q-0.027 0.071-0.143 0.143t-0.143 0.107q-0.045 0.071-0.013 0.165t0.094 0.147q0.018 0.018 0.013 0.036t-0.031 0.040-0.049 0.036-0.058 0.031l-0.027 0.018q-0.098 0.045-0.183-0.054t-0.121-0.232q-0.063-0.223-0.143-0.268-0.205-0.071-0.259 0.009-0.045-0.116-0.366-0.232-0.223-0.080-0.518-0.036 0.054-0.009 0-0.134-0.063-0.134-0.17-0.107 0.027-0.054 0.036-0.156t0.009-0.121q0.027-0.116 0.107-0.205 0.009-0.009 0.063-0.076t0.085-0.121 0.004-0.054q0.313 0.036 0.446-0.098 0.045-0.045 0.103-0.152t0.094-0.152q0.080-0.054 0.125-0.049t0.129 0.049 0.129 0.045q0.125 0.009 0.138-0.098t-0.067-0.179q0.107 0.009 0.027-0.152-0.045-0.063-0.071-0.080-0.107-0.036-0.241 0.045-0.071 0.036 0.018 0.071-0.009-0.009-0.085 0.094t-0.147 0.156-0.143-0.045q-0.009-0.009-0.049-0.121t-0.085-0.121q-0.071 0-0.143 0.134 0.027-0.071-0.098-0.134t-0.214-0.071q0.17-0.107-0.071-0.241-0.063-0.036-0.183-0.045t-0.174 0.036q-0.045 0.063-0.049 0.103t0.045 0.071 0.094 0.049 0.103 0.036 0.076 0.027q0.125 0.089 0.071 0.125-0.018 0.009-0.076 0.031t-0.103 0.040-0.054 0.036q-0.027 0.036 0 0.125t-0.018 0.125q-0.045-0.045-0.080-0.156t-0.063-0.147q0.063 0.080-0.223 0.054l-0.089-0.009q-0.036 0-0.143 0.018t-0.183 0.009-0.121-0.071q-0.036-0.071 0-0.179 0.009-0.036 0.036-0.018-0.036-0.027-0.098-0.085t-0.089-0.076q-0.411 0.134-0.839 0.366 0.054 0.009 0.107-0.009 0.045-0.018 0.116-0.058t0.089-0.049q0.304-0.125 0.375-0.063l0.045-0.045q0.125 0.143 0.179 0.223-0.063-0.036-0.268-0.009-0.179 0.054-0.196 0.107 0.063 0.107 0.045 0.161-0.036-0.027-0.103-0.089t-0.129-0.098-0.134-0.045q-0.143 0-0.196 0.009-1.304 0.714-2.098 1.982zM8.83 13.482q0 0.054 0.018 0.143 1.839-0.321 3.134-1.688-0.027-0.027-0.112-0.040t-0.112-0.031q-0.161-0.063-0.214-0.071 0.009-0.063-0.022-0.116t-0.071-0.080-0.112-0.071-0.098-0.063q-0.018-0.018-0.063-0.054t-0.063-0.049-0.067-0.040-0.076-0.018-0.089 0.009l-0.027 0.009q-0.027 0.009-0.049 0.022t-0.049 0.027-0.036 0.027 0 0.022q-0.188-0.152-0.321-0.196-0.045-0.009-0.098-0.049t-0.094-0.063-0.089-0.013-0.103 0.063q-0.045 0.045-0.054 0.134t-0.018 0.116q-0.063-0.045 0-0.156t0.018-0.165q-0.027-0.054-0.094-0.040t-0.107 0.040-0.103 0.076-0.080 0.058-0.076 0.049-0.076 0.067q-0.027 0.036-0.054 0.107t-0.045 0.098q-0.018-0.036-0.103-0.058t-0.085-0.049q0.018 0.089 0.036 0.313t0.045 0.339q0.063 0.277-0.107 0.429-0.241 0.223-0.259 0.357-0.036 0.196 0.107 0.232 0 0.063-0.071 0.183t-0.063 0.192z"></path></symbol><symbol id="icon-snippet" viewBox="0 0 16 16"><path d="M5.858 13.212c-2.064-0.593-7.973-3.2-1.264-11.818 3.612 2.451 5.006 4.619 5.264 6.399-1.729 0.619-2.916 1.832-2.761 4.025-0.955-2.245-1.703-5.367-2.013-7.302-0.155 2.271 0.181 6.296 0.774 8.696zM8.155 13.754c-0.49-1.368-1.522-5.883 6.219-5.651 0.232 7.096-3.535 6.838-5.212 6.348 0.697-1.368 1.6-2.606 2.735-3.638-1.393 0.774-2.658 1.755-3.741 2.942z"></path></symbol><symbol id="icon-spinner" viewBox="0 0 16 16"><path d="M3.339 12.661c-1.236-1.236-1.931-2.91-1.931-4.661 0-3.631 2.961-6.592 6.592-6.592l0.824 0.824-0.824 0.824c-2.729 0-4.944 2.214-4.944 4.944 0 1.313 0.515 2.575 1.442 3.502l-1.004 0.129zM8 14.592l-0.824-0.824 0.824-0.824c2.729 0 4.944-2.214 4.944-4.944 0-1.313-0.515-2.575-1.442-3.502l1.004-0.129 0.155-1.030c1.236 1.236 1.931 2.91 1.931 4.661 0 3.631-2.961 6.592-6.592 6.592z"></path></symbol><symbol id="icon-strikethrough" viewBox="0 0 512 512"><path d="M496 224H293.9l-87.17-26.83A43.55 43.55 0 0 1 219.55 112h66.79A49.89 49.89 0 0 1 331 139.58a16 16 0 0 0 21.46 7.15l42.94-21.47a16 16 0 0 0 7.16-21.46l-.53-1A128 128 0 0 0 287.51 32h-68a123.68 123.68 0 0 0-123 135.64c2 20.89 10.1 39.83 21.78 56.36H16a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h480a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16zm-180.24 96A43 43 0 0 1 336 356.45 43.59 43.59 0 0 1 292.45 400h-66.79A49.89 49.89 0 0 1 181 372.42a16 16 0 0 0-21.46-7.15l-42.94 21.47a16 16 0 0 0-7.16 21.46l.53 1A128 128 0 0 0 224.49 480h68a123.68 123.68 0 0 0 123-135.64 114.25 114.25 0 0 0-5.34-24.36z"></path></symbol><symbol id="icon-subscript" viewBox="0 0 512 512"><path d="M496 448h-16V304a16 16 0 0 0-16-16h-48a16 16 0 0 0-14.29 8.83l-16 32A16 16 0 0 0 400 352h16v96h-16a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h96a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16zM336 64h-67a16 16 0 0 0-13.14 6.87l-79.9 115-79.9-115A16 16 0 0 0 83 64H16A16 16 0 0 0 0 80v48a16 16 0 0 0 16 16h33.48l77.81 112-77.81 112H16a16 16 0 0 0-16 16v48a16 16 0 0 0 16 16h67a16 16 0 0 0 13.14-6.87l79.9-115 79.9 115A16 16 0 0 0 269 448h67a16 16 0 0 0 16-16v-48a16 16 0 0 0-16-16h-33.48l-77.81-112 77.81-112H336a16 16 0 0 0 16-16V80a16 16 0 0 0-16-16z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 16"><path d="M8 0.969c-3.873 0-7.031 3.159-7.031 7.031s3.159 7.031 7.031 7.031c3.873 0 7.031-3.159 7.031-7.031s-3.159-7.031-7.031-7.031zM6.901 11.241l-3.213-3.241 1.236-1.236 1.978 2.005 4.23-4.23 1.236 1.236z"></path></symbol><symbol id="icon-superscript" viewBox="0 0 512 512"><path d="M496 160h-16V16a16 16 0 0 0-16-16h-48a16 16 0 0 0-14.29 8.83l-16 32A16 16 0 0 0 400 64h16v96h-16a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h96a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16zM336 64h-67a16 16 0 0 0-13.14 6.87l-79.9 115-79.9-115A16 16 0 0 0 83 64H16A16 16 0 0 0 0 80v48a16 16 0 0 0 16 16h33.48l77.81 112-77.81 112H16a16 16 0 0 0-16 16v48a16 16 0 0 0 16 16h67a16 16 0 0 0 13.14-6.87l79.9-115 79.9 115A16 16 0 0 0 269 448h67a16 16 0 0 0 16-16v-48a16 16 0 0 0-16-16h-33.48l-77.81-112 77.81-112H336a16 16 0 0 0 16-16V80a16 16 0 0 0-16-16z"></path></symbol><symbol id="icon-table" viewBox="0 0 15 16"><path d="M4.571 12.286v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM4.571 8.857v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM9.143 12.286v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM4.571 5.429v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM9.143 8.857v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM13.714 12.286v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM9.143 5.429v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM13.714 8.857v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM13.714 5.429v-1.714q0-0.125-0.080-0.205t-0.205-0.080h-2.857q-0.125 0-0.205 0.080t-0.080 0.205v1.714q0 0.125 0.080 0.205t0.205 0.080h2.857q0.125 0 0.205-0.080t0.080-0.205zM14.857 2.571v9.714q0 0.589-0.42 1.009t-1.009 0.42h-12q-0.589 0-1.009-0.42t-0.42-1.009v-9.714q0-0.589 0.42-1.009t1.009-0.42h12q0.589 0 1.009 0.42t0.42 1.009z"></path></symbol><symbol id="icon-tag" viewBox="0 0 16 16"><path d="M5.156 4c0-0.313-0.125-0.594-0.344-0.813s-0.5-0.344-0.813-0.344c-0.313 0-0.594 0.125-0.813 0.344s-0.344 0.5-0.344 0.813c0 0.313 0.125 0.594 0.344 0.813s0.5 0.344 0.813 0.344c0.313 0 0.594-0.125 0.813-0.344s0.344-0.5 0.344-0.813zM14.656 9.156c0 0.313-0.094 0.563-0.313 0.781l-4.375 4.406c-0.25 0.219-0.531 0.313-0.813 0.313-0.344 0-0.594-0.094-0.813-0.313l-6.375-6.406c-0.25-0.219-0.438-0.5-0.594-0.906-0.156-0.375-0.219-0.719-0.219-1.031v-3.719c0-0.313 0.094-0.563 0.313-0.813 0.25-0.219 0.5-0.313 0.813-0.313h3.719c0.313 0 0.656 0.063 1.031 0.219 0.406 0.156 0.688 0.344 0.938 0.594l6.375 6.375c0.219 0.219 0.313 0.5 0.313 0.813z"></path></symbol><symbol id="icon-tasks" viewBox="0 0 560 576"><path d="M139.61 35.5a12 12 0 0 0-17 0L58.93 98.81l-22.7-22.12a12 12 0 0 0-17 0L3.53 92.41a12 12 0 0 0 0 17l47.59 47.4a12.78 12.78 0 0 0 17.61 0l15.59-15.62L156.52 69a12.09 12.09 0 0 0 .09-17zm0 159.19a12 12 0 0 0-17 0l-63.68 63.72-22.7-22.1a12 12 0 0 0-17 0L3.53 252a12 12 0 0 0 0 17L51 316.5a12.77 12.77 0 0 0 17.6 0l15.7-15.69 72.2-72.22a12 12 0 0 0 .09-16.9zM64 368c-26.49 0-48.59 21.5-48.59 48S37.53 464 64 464a48 48 0 0 0 0-96zm432 16H208a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h288a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16zm0-320H208a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h288a16 16 0 0 0 16-16V80a16 16 0 0 0-16-16zm0 160H208a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h288a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16z"></path></symbol><symbol id="icon-thumbtack" viewBox="0 0 384 512"><path d="M298.028 214.267L285.793 96H328c13.255 0 24-10.745 24-24V24c0-13.255-10.745-24-24-24H56C42.745 0 32 10.745 32 24v48c0 13.255 10.745 24 24 24h42.207L85.972 214.267C37.465 236.82 0 277.261 0 328c0 13.255 10.745 24 24 24h136v104.007c0 1.242.289 2.467.845 3.578l24 48c2.941 5.882 11.364 5.893 14.311 0l24-48a8.008 8.008 0 0 0 .845-3.578V352h136c13.255 0 24-10.745 24-24-.001-51.183-37.983-91.42-85.973-113.733z"></path></symbol><symbol id="icon-tick-inverse" viewBox="0 0 16 16"><path d="M7.25 11.594l5.5-5.469c0.094-0.125 0.156-0.25 0.156-0.406s-0.063-0.281-0.156-0.406l-0.906-0.906c-0.125-0.125-0.25-0.188-0.406-0.188s-0.313 0.063-0.406 0.188l-4.188 4.156-1.875-1.875c-0.094-0.125-0.25-0.156-0.406-0.156s-0.281 0.031-0.406 0.156l-0.906 0.906c-0.094 0.125-0.156 0.25-0.156 0.406s0.063 0.281 0.156 0.406l3.219 3.188c0.094 0.125 0.219 0.188 0.375 0.188s0.313-0.063 0.406-0.188zM14.844 3.719v8.563c0 0.719-0.25 1.313-0.75 1.813s-1.094 0.75-1.813 0.75h-8.563c-0.719 0-1.313-0.25-1.813-0.75s-0.75-1.094-0.75-1.813v-8.563c0-0.719 0.25-1.313 0.75-1.813s1.094-0.75 1.813-0.75h8.563c0.719 0 1.313 0.25 1.813 0.75s0.75 1.094 0.75 1.813z"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="M14.906 4.563c0 0.219-0.063 0.438-0.25 0.594l-7.656 7.688c-0.188 0.156-0.375 0.25-0.625 0.25-0.219 0-0.438-0.094-0.594-0.25l-4.438-4.438c-0.188-0.188-0.25-0.375-0.25-0.625 0-0.219 0.063-0.438 0.25-0.594l1.188-1.219c0.188-0.156 0.375-0.25 0.625-0.25 0.219 0 0.438 0.094 0.594 0.25l2.625 2.625 5.875-5.875c0.156-0.156 0.375-0.25 0.594-0.25 0.25 0 0.438 0.094 0.625 0.25l1.188 1.219c0.188 0.188 0.25 0.375 0.25 0.625z"></path></symbol><symbol id="icon-time" viewBox="0 0 16 16"><path d="M9.889 11.055l-2.713-2.713v-3.637h1.648v2.955l2.231 2.231zM8 1.408c-3.641 0-6.592 2.951-6.592 6.592s2.951 6.592 6.592 6.592 6.592-2.951 6.592-6.592-2.951-6.592-6.592-6.592zM8 12.944c-2.73 0-4.944-2.213-4.944-4.944s2.213-4.944 4.944-4.944c2.73 0 4.944 2.213 4.944 4.944s-2.213 4.944-4.944 4.944z"></path></symbol><symbol id="icon-title" viewBox="0 0 16 16"><path d="M0 4.58v-3.42l0.723-0.009 0.482 0.241q0.107 0.045 1.884 0.045 0.393 0 1.179-0.018t1.179-0.018q0.321 0 0.96 0.004t0.96 0.004h2.616q0.054 0 0.188 0.004t0.183 0 0.143-0.027 0.156-0.080 0.134-0.156l0.375-0.009q0.036 0 0.125 0.004t0.125 0.004q0.018 1 0.018 3 0 0.714-0.045 0.973-0.348 0.125-0.607 0.161-0.223-0.393-0.482-1.143-0.027-0.080-0.098-0.429t-0.129-0.656-0.067-0.317q-0.054-0.071-0.107-0.112t-0.138-0.054-0.116-0.022-0.161-0.004-0.147 0.004q-0.152 0-0.594-0.004t-0.665-0.004-0.571 0.018-0.634 0.054q-0.080 0.723-0.071 1.214 0 0.839 0.018 3.464t0.018 4.063q0 0.143-0.022 0.638t0 0.817 0.112 0.616q0.357 0.188 1.107 0.379t1.071 0.335q0.045 0.357 0.045 0.446 0 0.125-0.027 0.259l-0.304 0.009q-0.679 0.018-1.946-0.071t-1.848-0.089q-0.446 0-1.348 0.080t-1.357 0.080q-0.027-0.455-0.027-0.464v-0.080q0.152-0.241 0.549-0.384t0.879-0.259 0.696-0.241q0.17-0.375 0.17-3.42 0-0.902-0.027-2.705t-0.027-2.705v-1.045q0-0.018 0.004-0.138t0.004-0.223-0.009-0.228-0.027-0.214-0.045-0.125q-0.098-0.107-1.446-0.107-0.295 0-0.83 0.107t-0.714 0.232q-0.17 0.116-0.304 0.647t-0.281 0.991-0.379 0.478q-0.375-0.232-0.5-0.393zM12.625 12.737q0.080-0.165 0.375-0.165h0.714v-9.143h-0.714q-0.295 0-0.375-0.165t0.098-0.397l1.125-1.446q0.179-0.232 0.438-0.232t0.438 0.232l1.125 1.446q0.179 0.232 0.098 0.397t-0.375 0.165h-0.714v9.143h0.714q0.295 0 0.375 0.165t-0.098 0.397l-1.125 1.446q-0.179 0.232-0.438 0.232t-0.438-0.232l-1.125-1.446q-0.179-0.232-0.098-0.397z"></path></symbol><symbol id="icon-undo" viewBox="0 0 16 16"><path d="M14.416 8c0 0.879-0.146 1.699-0.498 2.49s-0.791 1.494-1.377 2.051c-0.557 0.586-1.26 1.025-2.051 1.377s-1.611 0.498-2.49 0.498c-0.967 0-1.875-0.176-2.725-0.586-0.879-0.41-1.611-0.996-2.227-1.729-0.029-0.059-0.059-0.117-0.059-0.176 0-0.088 0.029-0.117 0.088-0.176l1.143-1.143c0.059-0.059 0.117-0.088 0.205-0.088s0.146 0.059 0.205 0.088c0.381 0.557 0.908 0.967 1.494 1.23 0.586 0.293 1.201 0.439 1.875 0.439 0.586 0 1.143-0.117 1.67-0.322 0.527-0.234 0.967-0.527 1.348-0.938 0.41-0.381 0.703-0.82 0.938-1.348 0.205-0.527 0.322-1.084 0.322-1.67s-0.117-1.143-0.322-1.67c-0.234-0.527-0.527-0.967-0.938-1.348-0.381-0.41-0.82-0.703-1.348-0.938-0.527-0.205-1.084-0.322-1.67-0.322-0.557 0-1.084 0.088-1.582 0.293s-0.938 0.469-1.318 0.85l1.143 1.143c0.176 0.176 0.205 0.381 0.117 0.586-0.117 0.234-0.264 0.322-0.498 0.322h-3.75c-0.146 0-0.264-0.029-0.381-0.146s-0.146-0.234-0.146-0.381v-3.75c0-0.234 0.088-0.381 0.322-0.498 0.205-0.088 0.41-0.059 0.586 0.117l1.084 1.084c0.586-0.557 1.289-0.996 2.051-1.318 0.762-0.293 1.553-0.439 2.373-0.439 0.879 0 1.699 0.146 2.49 0.498s1.494 0.791 2.051 1.377c0.586 0.557 1.025 1.26 1.377 2.051s0.498 1.611 0.498 2.49z"></path></symbol><symbol id="icon-uni52" viewBox="0 0 16 16"><path d="M13.75 3.844h-5.875c-0.313-0.375-0.625-0.781-0.719-0.875-0.063-0.188-0.25-0.313-0.438-0.313h-2.344c-0.156 0-0.281 0.094-0.406 0.188l-0.781 1h-0.938c-0.5 0-0.875 0.406-0.875 0.875v7.75c0 0.469 0.375 0.875 0.875 0.875h11.5c0.5 0 0.875-0.406 0.875-0.875v-7.75c0-0.469-0.375-0.875-0.875-0.875z"></path></symbol><symbol id="icon-upload" viewBox="0 0 512 512"><path d="M296 384h-80c-13.3 0-24-10.7-24-24V192h-87.7c-17.8 0-26.7-21.5-14.1-34.1L242.3 5.7c7.5-7.5 19.8-7.5 27.3 0l152.2 152.2c12.6 12.6 3.7 34.1-14.1 34.1H320v168c0 13.3-10.7 24-24 24zm216-8v112c0 13.3-10.7 24-24 24H24c-13.3 0-24-10.7-24-24V376c0-13.3 10.7-24 24-24h136v8c0 30.9 25.1 56 56 56h80c30.9 0 56-25.1 56-56v-8h136c13.3 0 24 10.7 24 24zm-124 88c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20zm64 0c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20z"></path></symbol><symbol id="icon-user" viewBox="0 0 16 16"><path d="M1.798 12.261q0-0.444 0.029-0.866t0.117-0.912 0.222-0.908 0.36-0.816 0.519-0.678 0.716-0.448 0.933-0.167q0.075 0 0.352 0.18t0.624 0.402 0.904 0.402 1.117 0.18 1.117-0.18 0.904-0.402 0.624-0.402 0.352-0.18q0.511 0 0.933 0.167t0.716 0.448 0.519 0.678 0.36 0.816 0.222 0.908 0.117 0.912 0.029 0.866q0 1.004-0.611 1.586t-1.624 0.582h-7.316q-1.013 0-1.624-0.582t-0.611-1.586zM4.477 4.786q0-1.331 0.942-2.273t2.273-0.942 2.273 0.942 0.942 2.273-0.942 2.273-2.273 0.942-2.273-0.942-0.942-2.273z"></path></symbol><symbol id="icon-view" viewBox="0 0 16 16"><path d="M0.969 8.002q0-0.267 0.157-0.541 1.099-1.797 2.955-2.888t3.92-1.091 3.92 1.091 2.955 2.888q0.157 0.275 0.157 0.541t-0.157 0.541q-1.099 1.805-2.955 2.892t-3.92 1.087-3.92-1.091-2.955-2.888q-0.157-0.275-0.157-0.541zM1.973 8.002q1.044 1.609 2.617 2.562t3.41 0.953 3.41-0.953 2.617-2.562q-1.193-1.852-2.99-2.77 0.479 0.816 0.479 1.766 0 1.452-1.032 2.484t-2.484 1.032-2.484-1.032-1.032-2.484q0-0.95 0.479-1.766-1.797 0.918-2.99 2.77zM5.614 6.998q0 0.157 0.11 0.267t0.267 0.11 0.267-0.11 0.11-0.267q0-0.675 0.479-1.154t1.154-0.479q0.157 0 0.267-0.11t0.11-0.267-0.11-0.267-0.267-0.11q-0.981 0-1.683 0.702t-0.702 1.683z"></path></symbol><symbol id="icon-wagtail-inverse" viewBox="0 0 16 16"><path d="M10.637 6.096c0 0.059 0 0.117 0 0.176s-0.029 0.117-0.029 0.205c0 0.059-0.029 0.146-0.029 0.205-0.029 0.088-0.059 0.176-0.059 0.264-0.029 0.088-0.059 0.176-0.088 0.264s-0.059 0.176-0.088 0.293c-0.029 0.029-0.059 0.088-0.059 0.117-0.029 0.029-0.029 0.059-0.029 0.088-0.029 0.029-0.029 0.029-0.059 0.059-0.029 0.059-0.029 0.088-0.059 0.146s-0.059 0.088-0.088 0.146l-0.059 0.059-0.029 0.088c-0.029 0.029-0.059 0.088-0.117 0.117-0.059 0.088-0.117 0.205-0.205 0.264-0.088 0.088-0.176 0.176-0.264 0.264l-0.117 0.117c-0.059 0.029-0.088 0.059-0.146 0.117-0.176 0.146-0.381 0.264-0.586 0.381s-0.41 0.205-0.615 0.293c-0.205 0.088-0.41 0.176-0.615 0.234-0.176 0.059-0.352 0.117-0.527 0.176-0.352 0.088-0.645 0.146-0.85 0.205s-0.322 0.059-0.322 0.059c0 0 0.117 0 0.322-0.059 0.205-0.029 0.498-0.088 0.85-0.176 0.176-0.059 0.352-0.088 0.557-0.146s0.41-0.146 0.615-0.234c0.205-0.088 0.41-0.176 0.615-0.293 0.205-0.088 0.41-0.234 0.615-0.381 0.059-0.029 0.088-0.059 0.146-0.088l0.146-0.117c0.088-0.088 0.176-0.176 0.234-0.264 0.088-0.088 0.176-0.176 0.234-0.264 0.059-0.059 0.088-0.088 0.117-0.146l0.059-0.059c0-0.029 0.029-0.059 0.029-0.088 0.029-0.029 0.059-0.088 0.117-0.146 0-0.029 0.029-0.088 0.059-0.146 0.029 0 0.029-0.029 0.059-0.059 0-0.029 0-0.059 0.029-0.088s0.029-0.088 0.059-0.146c0.059-0.088 0.088-0.176 0.117-0.264 0.029-0.117 0.059-0.205 0.088-0.293s0.029-0.176 0.059-0.234c0.029-0.088 0.029-0.176 0.059-0.234 0-0.088 0-0.146 0.029-0.205 0-0.088 0-0.146 0-0.176 0.029-0.234 0.029-0.352 0.029-0.352h-0.176c0 0 0 0.117-0.029 0.322zM13.918 3.752l-0.996-0.967c-0.029-0.967-0.645-1.611-0.938-1.787-0.293-0.264-0.967-0.469-1.729-0.381-0.732 0.117-1.787 0.498-2.52 1.875-1.436 2.695-4.512 8.467-4.541 8.525-0.029 0-0.029 0.029 0 0.029 0 0.029 0.029 0.029 0.029 0.029l1.172-0.205-2.285 4.424c0 0.029 0 0.029 0.029 0.059 0 0 0 0 0.029 0 0 0 0 0 0 0l1.641-0.293c0 0 0.029 0 0.029-0.029l1.201-3.516c3.076-0.029 5.215-0.762 6.387-2.256 1.23-1.582 1.055-3.574 0.85-4.512l-0.146-0.908h1.758c0 0 0.029 0 0.029-0.029 0.029 0 0-0.029 0-0.059zM12.307 2.521c0 0.146-0.146 0.293-0.293 0.293-0.176 0-0.293-0.146-0.293-0.293 0-0.176 0.117-0.322 0.293-0.322 0.146 0 0.293 0.146 0.293 0.322zM11.34 9.201c-1.172 1.465-3.281 2.197-6.328 2.197-0.029 0-0.029 0.029-0.029 0.059l-1.23 3.516-1.494 0.264 2.285-4.424c0-0.029 0-0.029 0-0.059-0.029 0-0.029-0.029-0.059-0.029l-1.172 0.234c0.41-0.762 3.193-5.918 4.512-8.408 0.732-1.348 1.729-1.729 2.432-1.816 0.732-0.088 1.348 0.088 1.641 0.293v0.645c-0.059 0-0.469-0.146-0.85-0.146s-0.82 0.146-1.084 0.439c-0.469 0.527-0.586 1.055-0.352 1.934 1.582-0.293 2.285 0.381 2.549 0.791 0.205 0.879 0.439 2.9-0.82 4.512zM12.189 3.752l0.674-0.879 0.879 0.879z"></path></symbol><symbol id="icon-wagtail-icon" viewBox="0 0 16 16"><path d="M12.324 2.539c0 0.152-0.114 0.303-0.303 0.303-0.152 0-0.303-0.152-0.303-0.303 0-0.19 0.152-0.303 0.303-0.303 0.19 0 0.303 0.114 0.303 0.303zM12.437 5.421c0 0-0.417-1.896-2.92-1.403-0.228-0.872-0.19-1.517 0.341-2.086 0.759-0.796 1.972-0.379 1.972-0.379v-0.645c-0.417-0.19-0.834-0.228-1.252-0.228-1.631 0-2.541 1.214-2.92 2.048l-4.513 8.306 1.289-0.228-2.351 4.513 1.631-0.303 1.252-3.527c3.527 0 8.078-1.289 7.471-6.068zM13.954 3.828l-0.986-0.986-0.796 0.986zM5.535 10.465c0 0 0.152-0.038 0.341-0.076 0.19-0.076 0.493-0.114 0.834-0.228 0.19-0.038 0.379-0.114 0.569-0.152 0.19-0.076 0.417-0.152 0.607-0.228 0.228-0.114 0.417-0.19 0.607-0.303 0.228-0.114 0.417-0.265 0.607-0.379 0.038-0.038 0.114-0.076 0.152-0.114l0.114-0.114c0.114-0.114 0.19-0.19 0.265-0.265s0.152-0.19 0.228-0.265c0.038-0.038 0.076-0.076 0.114-0.152l0.038-0.076 0.038-0.038c0.038-0.076 0.076-0.114 0.076-0.152 0.038-0.038 0.076-0.114 0.076-0.152 0.038-0.038 0.038-0.038 0.038-0.076 0.038 0 0.038-0.038 0.038-0.076 0.038-0.038 0.038-0.076 0.076-0.114 0.038-0.114 0.076-0.19 0.114-0.303 0-0.076 0.038-0.152 0.076-0.265 0-0.076 0.038-0.152 0.038-0.228s0.038-0.152 0.038-0.228c0-0.076 0.038-0.152 0.038-0.228 0-0.038 0-0.114 0-0.152 0-0.19 0.038-0.341 0.038-0.341l0.19 0.038c0 0-0.038 0.114-0.038 0.303 0 0.076-0.038 0.114-0.038 0.19s0 0.152-0.038 0.19c0 0.076 0 0.152-0.038 0.265 0 0.076-0.038 0.152-0.038 0.228-0.038 0.114-0.076 0.19-0.114 0.303-0.038 0.076-0.076 0.19-0.114 0.265 0 0.038-0.038 0.114-0.076 0.152 0 0.038 0 0.038-0.038 0.076 0 0.038 0 0.038-0.038 0.076 0 0.038-0.038 0.076-0.076 0.152-0.038 0.038-0.038 0.076-0.076 0.114s-0.038 0.076-0.038 0.076l-0.076 0.076c-0.038 0.038-0.076 0.114-0.114 0.152-0.076 0.076-0.152 0.19-0.228 0.265s-0.152 0.152-0.265 0.265l-0.114 0.114c-0.076 0.038-0.114 0.038-0.152 0.076-0.228 0.152-0.417 0.303-0.645 0.417-0.19 0.076-0.417 0.19-0.607 0.265-0.228 0.076-0.417 0.152-0.645 0.228-0.19 0.038-0.379 0.114-0.569 0.152-0.341 0.076-0.645 0.152-0.834 0.19s-0.341 0.076-0.341 0.076z"></path></symbol><symbol id="icon-warning" viewBox="0 0 16 16"><path d="M14.674 13.246c0.11 0.165 0.11 0.33 0 0.494-0.082 0.137-0.22 0.22-0.412 0.22 0 0-12.552 0-12.552 0-0.165 0-0.302-0.082-0.385-0.22-0.11-0.165-0.11-0.33-0.027-0.494 0 0 6.262-10.986 6.262-10.986 0.082-0.165 0.22-0.247 0.439-0.247 0.192 0 0.33 0.082 0.412 0.247 0 0 6.262 10.986 6.262 10.986zM8.769 12.559c0 0 0-1.401 0-1.401s-1.538 0-1.538 0c0 0 0 1.401 0 1.401s1.538 0 1.538 0zM8.769 10.115c0 0 0-4.23 0-4.23s-1.538 0-1.538 0c0 0 0 4.23 0 4.23s1.538 0 1.538 0z"></path></symbol></defs></svg>
  )
}
```

Reply:
Thanks! The main reason I didn't go with JSX is that the sprite is massive and is completely static. So wasn't worth the time converting all the attributes and waiting for the whole thing to compile!

It's a temporary workaround. Ideally, we'd find a way to reference the files in Django, but the implementation is rather custom so would need some work to replicate it or change it all to use standard SVG files.